# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# giogio <g.sora@tiscali.it>, 2013-2014
# Martijn Dekker <mcdutchie@hotmail.com>, 2012
# Nik Kalach <nikka@fedoraproject.org>, 2012
# Nik Kalach <nikka@fedoraproject.org>, 2012, 2012-2013, 2013
# Nik Kalach <nik.kalach@inbox.ru>, 2012-2014
# Eduardo Vera <hmcb2080@gmail.com>, 2024.
# eduver <hmcb2080@gmail.com>, 2024.
msgid ""
msgstr ""
"Project-Id-Version: anaconda 32.22\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2020-03-17 14:21+0100\n"
"PO-Revision-Date: 2024-12-16 07:18+0000\n"
"Last-Translator: eduver <hmcb2080@gmail.com>\n"
"Language-Team: Interlingua <https://translate.fedoraproject.org/projects/"
"anaconda/f32/ia/>\n"
"Language: ia\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 5.8.4\n"

#: pyanaconda/ui/categories/system.py:28
#: pyanaconda/ui/gui/spokes/lib/accordion.py:458
msgid "SYSTEM"
msgstr "SYSTEMA"

#: pyanaconda/ui/categories/user_settings.py:28
msgid "USER SETTINGS"
msgstr "PARAMETROS DE CONFIGURATION DE USATOR"

#: pyanaconda/ui/categories/customization.py:28
msgid "CUSTOMIZATION"
msgstr "PERSONALISATION"

#: pyanaconda/ui/categories/__init__.py:41
msgid "DEFAULT TITLE"
msgstr "TITULO PREDEFINITE"

#: pyanaconda/ui/categories/localization.py:28
msgid "LOCALIZATION"
msgstr "LOCALISATION"

#: pyanaconda/ui/categories/software.py:28
msgid "SOFTWARE"
msgstr "SOFTWARE"

#: pyanaconda/ui/gui/__init__.py:415 widgets/src/BaseWindow.c:151
msgid "Help!"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:915
#: pyanaconda/ui/gui/spokes/lib/detailederror.py:58
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/__init__.py:931
msgctxt "GUI|Yes No Dialog"
msgid "_No"
msgstr "_No"

#: pyanaconda/ui/gui/__init__.py:932
msgctxt "GUI|Yes No Dialog"
msgid "_Yes"
msgstr "_Si"

#: pyanaconda/ui/gui/hubs/__init__.py:270
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""
"Completa le punctos marcate per iste icona ante de continuar con le sequente "
"passo."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:158
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the portal IP address <b>%(portalAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py:82
msgctxt "GUI|Advanced Storage|NVDIM"
msgid "No device to be reconfigured selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.py:174
msgid "Rescanning disks finished."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:49
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr "Crear un nove %(container_type)s ..."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:50
#, python-format
msgid "Create or select %(container_type)s"
msgstr "Crear o seliger %(container_type)s"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:51
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr "CONFIGURAR %(container_type)s"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:52
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""
"Crea un nomine pro iste %(container_type)s e seligie al minus un disco in "
"basso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:59
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:62
msgid "Volume Group"
msgstr "Gruppo de volumines"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:60
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:63
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Volume Group:"
msgstr "Gruppo de _volumines:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:65
msgid "Volume"
msgstr "Volumine"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:66
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Volume:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:204
msgid "container"
msgstr "container"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:205
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "container"
msgstr "contentor"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:335
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Do _not show this dialog for other selected file systems."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:341
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all file systems which are only used by {}."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:362
msgid ""
"{} may be a system boot partition! Deleting it may break other operating "
"systems. Are you sure you want to delete it?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:368
msgid ""
"Are you sure you want to delete all of the data on {}, including subvolumes?"
msgstr ""
"Es tu secur que tu vole deler tote le datos sur {}, includente le "
"subvolumines?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:373
msgid ""
"Are you sure you want to delete all of the data on {}, including snapshots?"
msgstr ""
"Es tu secur que tu vole deler tote le datos sur {}, includente le "
"instantaneos?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:376
msgid "Are you sure you want to delete all of the data on {}?"
msgstr "Es tu secur que tu vole deler tote le datos sur {}?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:592
#: pyanaconda/modules/storage/partitioning/interactive/utils.py:338
msgid "Name is already in use."
msgstr "Le nomine ja es in uso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:668
msgid "No disks selected."
msgstr "Nulle disco seligite."

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:48
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/cart.py:177
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:182
#: pyanaconda/ui/gui/spokes/lib/cart.glade:181
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr "_Definir como dispositivo de initio"

#: pyanaconda/ui/gui/spokes/lib/resize.py:53
msgid "Preserve"
msgstr "Preservar"

#: pyanaconda/ui/gui/spokes/lib/resize.py:54
msgid "Shrink"
msgstr "Reducer"

#: pyanaconda/ui/gui/spokes/lib/resize.py:55
msgid "Delete"
msgstr "Deler"

#: pyanaconda/ui/gui/spokes/lib/resize.py:96
#, python-format
msgid "Installation requires a total of <b>%s</b> for system data."
msgstr "Le installation require un total de <b>%s</b> pro le datos de systema."

#: pyanaconda/ui/gui/spokes/lib/resize.py:128
msgid "This device contains the installation source."
msgstr "Iste dispositivo contine le fonte de installation."

#: pyanaconda/ui/gui/spokes/lib/resize.py:148
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:155
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:236
#, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr "%(freeSize)s de %(devSize)s"

#: pyanaconda/ui/gui/spokes/lib/resize.py:246
msgid "Not resizeable"
msgstr "Non redimensionabile"

#: pyanaconda/ui/gui/spokes/lib/resize.py:281
msgid "Free space"
msgstr "Liberar spatio"

#: pyanaconda/ui/gui/spokes/lib/resize.py:306
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:317
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr "Spatio total seligite a recuperar: <b>%s</b>"

#: pyanaconda/ui/gui/spokes/lib/resize.py:546
#: pyanaconda/ui/gui/spokes/lib/resize.py:551
#: pyanaconda/ui/gui/spokes/lib/resize.glade:267
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr "Deler _toto"

#: pyanaconda/ui/gui/spokes/lib/resize.py:548
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr "Preservar _toto"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:449
msgid "DATA"
msgstr "DATOS"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:498
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""
"Il non ha ancora create nulle punctos de montage pro le installation de "
"%(product)s %(version)s. On pote:"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:508
msgctxt "GUI|Custom Partitioning|Autopart Page"
msgid "_Click here to create them automatically."
msgstr "_Clicca hic pro crear los automaticamente."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:534
msgid "Create new mount points by clicking the '+' button."
msgstr "Crear nove punctos de montage per cliccar sur le button '+'."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:542
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""
"O, assignar nove punctos de montage al partitiones existente post lor "
"selection in basso."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:547
msgctxt "GUI|Custom Partitioning|Autopart Page"
msgid "_New mount points will use the following partitioning scheme:"
msgstr "_Nove punctos de montage usara le sequente schema de partitiones:"

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:91
#: pyanaconda/ui/lib/entropy.py:95
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/root_password.py:55
msgctxt "GUI|Spoke"
msgid "_Root Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:127
msgid "Root password has been set."
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:167
msgid "Disabled, set password to enable."
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.py:169
#: pyanaconda/ui/tui/spokes/root_password.py:77
msgid "Root account is disabled."
msgstr "Le conto de superusator es disactivate."

#: pyanaconda/ui/gui/spokes/root_password.py:172
msgid "Root password is set"
msgstr "Le contrasigno de superusator es definite"

#: pyanaconda/ui/gui/spokes/root_password.py:174
msgid "Root password is not set"
msgstr "Le contrasigno de superusator non es definite"

#: pyanaconda/ui/gui/spokes/installation_progress.py:98
msgid "Complete!"
msgstr "Complete!"

#: pyanaconda/ui/gui/spokes/installation_progress.py:120
#, python-format
msgid "Use of this product is subject to the license agreement found at %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_progress.py:145
msgctxt "GUI|Progress"
msgid "_Reboot System"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_progress.py:147
msgctxt "GUI|Progress"
msgid "_Finish Installation"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_progress.py:155
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot your system to start using it!"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_progress.py:160
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and quit the application to start using it!"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:67
msgid "Invalid group name."
msgstr "Le nomine del gruppo invalide."

#: pyanaconda/ui/gui/spokes/user.py:231
msgctxt "GUI|Spoke"
msgid "_User Creation"
msgstr "Creation de _usator"

#: pyanaconda/ui/gui/spokes/user.py:355
msgid "The password was set by kickstart."
msgstr "Le contrasigno esseva configurate per kickstart."

#: pyanaconda/ui/gui/spokes/user.py:445 pyanaconda/ui/tui/spokes/user.py:227
msgid "No user will be created"
msgstr "Nulle usator essera create"

#: pyanaconda/ui/gui/spokes/user.py:447 pyanaconda/ui/tui/spokes/user.py:231
#, python-format
msgid "Administrator %s will be created"
msgstr "Administrator %s essera create"

#: pyanaconda/ui/gui/spokes/user.py:449 pyanaconda/ui/tui/spokes/user.py:233
#, python-format
msgid "User %s will be created"
msgstr "Usator %s essera create"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:422
msgctxt "GUI|Spoke"
msgid "_Time & Date"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:559
msgid "Restoring hardware time..."
msgstr "Restablimento de tempore del hardware…"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:573
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:579
#: pyanaconda/ui/tui/spokes/time_spoke.py:202
#, python-format
msgid "%s timezone"
msgstr "Fuso horari %s"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:575
msgid "Invalid timezone"
msgstr "Fuso horari incorrecte"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:581
#: pyanaconda/ui/gui/spokes/installation_source.py:682
#: pyanaconda/ui/gui/spokes/software_selection.py:316
#: pyanaconda/ui/gui/spokes/software_selection.py:321
#: pyanaconda/ui/tui/spokes/installation_source.py:125
#: pyanaconda/ui/tui/spokes/software_selection.py:171
msgid "Nothing selected"
msgstr "Nihil seligite"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:749
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:763
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:949
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:749
msgid "PM"
msgstr "PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:751
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:760
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:948
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:951
msgid "AM"
msgstr "AM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1118
msgid "You need to set up networking first if you want to use NTP"
msgstr "Le connexion al rete debe esser configurate ante le uso de NTP"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1122
msgid "You have no working NTP server configured"
msgstr "Il non ha configurate nulle servitores NTP operante"

#: pyanaconda/ui/gui/spokes/network.py:81
msgid "Status unknown (missing)"
msgstr "Stato incognite (absente)"

#: pyanaconda/ui/gui/spokes/network.py:84
msgid "Status unknown"
msgstr "Stato incognite"

#: pyanaconda/ui/gui/spokes/network.py:86
msgid "Unmanaged"
msgstr "Non gerite"

#: pyanaconda/ui/gui/spokes/network.py:89
#: pyanaconda/ui/gui/spokes/network.py:93
msgid "Unavailable"
msgstr "Nondisponibile"

#: pyanaconda/ui/gui/spokes/network.py:91
msgid "Firmware missing"
msgstr "Firmware mancante"

#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Cable unplugged"
msgstr "Cablo disconnectite"

#: pyanaconda/ui/gui/spokes/network.py:99
msgid "Disconnected"
msgstr "Disconnectite"

#: pyanaconda/ui/gui/spokes/network.py:104
msgid "Connecting"
msgstr "Connectente"

#: pyanaconda/ui/gui/spokes/network.py:106
msgid "Authentication required"
msgstr "Authentication necessari"

#: pyanaconda/ui/gui/spokes/network.py:108
msgid "Connected"
msgstr "Connectite"

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Disconnecting"
msgstr "Disconnectente"

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Connection failed"
msgstr "Connexion fallite"

#: pyanaconda/ui/gui/spokes/network.py:247
#: pyanaconda/ui/gui/spokes/custom_storage.py:453
#: pyanaconda/ui/gui/spokes/custom_storage.py:485 pyanaconda/network.py:462
msgid "Unknown"
msgstr "Incognite"

#: pyanaconda/ui/gui/spokes/network.py:248
msgid "Ethernet"
msgstr "Ethernet"

#: pyanaconda/ui/gui/spokes/network.py:249
msgid "Wireless"
msgstr "Sin filo"

#: pyanaconda/ui/gui/spokes/network.py:250
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr "Bond"

#: pyanaconda/ui/gui/spokes/network.py:251
#: pyanaconda/ui/gui/spokes/network.glade:27
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:252
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Team"
msgstr "Equipa"

#: pyanaconda/ui/gui/spokes/network.py:253
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:769
msgid "unplugged"
msgstr "disconnectite"

#: pyanaconda/ui/gui/spokes/network.py:841
#: pyanaconda/ui/gui/spokes/network.glade:525
#: pyanaconda/ui/gui/spokes/network.glade:804
msgid "IPv4 Address"
msgstr "Adresse IPv4"

#: pyanaconda/ui/gui/spokes/network.py:842
#: pyanaconda/ui/gui/spokes/network.glade:496
#: pyanaconda/ui/gui/spokes/network.glade:816
#: pyanaconda/ui/gui/spokes/network.glade:1383
msgid "IPv6 Address"
msgstr "Adresse IPv6"

#: pyanaconda/ui/gui/spokes/network.py:844
#: pyanaconda/ui/gui/spokes/network.py:846
#: pyanaconda/ui/gui/spokes/network.glade:1371
msgid "IP Address"
msgstr "Adresse IP"

#: pyanaconda/ui/gui/spokes/network.py:940
#, python-format
msgid "%d Mb/s"
msgstr "%d Mb/s"

#: pyanaconda/ui/gui/spokes/network.py:1119
msgid "WEP"
msgstr "WEP"

#: pyanaconda/ui/gui/spokes/network.py:1122
msgid "WPA"
msgstr "WPA"

#: pyanaconda/ui/gui/spokes/network.py:1125
msgid "WPA2"
msgstr "WPA2"

#: pyanaconda/ui/gui/spokes/network.py:1129
msgid "Enterprise"
msgstr "Interprisa"

#: pyanaconda/ui/gui/spokes/network.py:1134 widgets/src/SpokeSelector.c:81
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:538
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:152
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:594
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:928
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1248
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1538
msgid "None"
msgstr "Nulle"

#: pyanaconda/ui/gui/spokes/network.py:1316
#: pyanaconda/ui/gui/spokes/network.glade:2452
msgid "Authentication required by wireless network"
msgstr "Authentication es requirite per rete sin filo"

#: pyanaconda/ui/gui/spokes/network.py:1317
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""
"Contrasignos o claves cryptographic es necessari \n"
"pro connecter se al rete sin filo '%(network_id)s'."

#: pyanaconda/ui/gui/spokes/network.py:1331
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr "Contra_signo:"

#: pyanaconda/ui/gui/spokes/network.py:1339
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr "_Clave:"

#: pyanaconda/ui/gui/spokes/network.py:1349
msgid "User name: "
msgstr "Nomine de usator: "

#: pyanaconda/ui/gui/spokes/network.py:1354
#: pyanaconda/ui/tui/spokes/askvnc.py:148
msgid "Password: "
msgstr "Contrasigno: "

#: pyanaconda/ui/gui/spokes/network.py:1360
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1365
msgid "Private key password: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1426
msgctxt "GUI|Spoke"
msgid "_Network & Host Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1530
#: pyanaconda/ui/gui/spokes/network.py:1549
#: pyanaconda/ui/gui/spokes/network.py:1642
#: pyanaconda/ui/gui/spokes/network.py:1660
#: pyanaconda/ui/tui/spokes/network.py:425
#, python-format
msgid "Host name is not valid: %s"
msgstr "Nomine de hospite invalide; %s"

#: pyanaconda/ui/gui/spokes/advanced_storage.py:510
msgctxt "GUI|Spoke"
msgid "_Installation Destination"
msgstr "Destination de _installation"

#: pyanaconda/ui/gui/spokes/advanced_storage.py:620
msgctxt "GUI|Installation Destination|Filter"
msgid "{} _storage device selected"
msgid_plural "{} _storage devices selected"
msgstr[0] "{} di_spositivo de immagazinage selectionate"
msgstr[1] "{} di_spositivos de immagazinage selectionate"

#: pyanaconda/ui/gui/spokes/custom_storage.py:101
#: pyanaconda/ui/gui/spokes/custom_storage.glade:87
msgid "MANUAL PARTITIONING"
msgstr "PARTITIONAMENTO MANUAL"

#: pyanaconda/ui/gui/spokes/custom_storage.py:273
#, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d di_spositovo de immagazinage seligite"
msgstr[1] "%d di_spositovos de immagazinage seligite"

#: pyanaconda/ui/gui/spokes/custom_storage.py:345
#, python-format
msgid "%(items_selected)s of %(items_total)s mount points in %(page_name)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:372
#, python-format
msgid ""
"Please select a single mount point to edit properties.\n"
"\n"
"You have currently selected:\n"
"%s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:377
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""
"Quando tu crea le punctos de montage pro le installation de %(name)s "
"%(version)s, lor detalios se monstrara hic."

#: pyanaconda/ui/gui/spokes/custom_storage.py:581
#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:178
msgid "Device reconfiguration failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:728
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:742
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and {} other"
msgid_plural " and {} others"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:789
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:818
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:823
msgid "This file system may not be resized."
msgstr "Iste systema de files non se pote redimensionar."

#: pyanaconda/ui/gui/spokes/custom_storage.py:916
msgid ""
"Error checking storage configuration. <a href=\"\">Click for details</a> or "
"press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:920
msgid ""
"Warning checking storage configuration. <a href=\"\">Click for details</a> "
"or press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1000
msgid "Failed to add new device."
msgstr "Impossibile adder un nove dispositivo."

#: pyanaconda/ui/gui/spokes/custom_storage.py:1057
msgid "Device removal request failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1191
msgid "No disks selected. Keeping previous disk set."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1269
#, fuzzy
#| msgid "(%s free)"
msgid "({} free)"
msgstr "(%s libere)"

#: pyanaconda/ui/gui/spokes/custom_storage.py:1456
msgid "Automatic partitioning failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1748
#: pyanaconda/ui/gui/spokes/custom_storage.py:1752
#: pyanaconda/ui/gui/spokes/installation_source.py:96
msgid " <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1761
#: pyanaconda/ui/gui/spokes/blivet_gui.py:318
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1772
#: pyanaconda/ui/gui/spokes/blivet_gui.py:326
msgctxt "GUI|Custom Partitioning|Reset Dialog"
msgid "_Reset selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1774
#: pyanaconda/ui/gui/spokes/blivet_gui.py:327
msgctxt "GUI|Custom Partitioning|Reset Dialog"
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.py:1851
msgid "Failed to unlock encrypted block device."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:104
msgid "Please wait... software metadata still loading."
msgstr ""
"Attende... le cargamento de metadatos de software es ancora in processo."

#: pyanaconda/ui/gui/spokes/storage.py:107
#, python-format
msgid ""
"Your current <b>%(product)s</b> software selection requires <b>%(total)s</b> "
"of available space, including <b>%(software)s</b> for software and <b>"
"%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:116
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:167
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""
"%s In le discos que ha seligite es le sequente amontas de spatio libere:"

#: pyanaconda/ui/gui/spokes/storage.py:177
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""
"<b>Le spatio pro le installation de %s es insufficiente</b>. On pote reducer "
"o eliminar le partitiones existente via le guida pro recuperar spatio, o on "
"pote adjustar le partitiones via le interfacie del partitionamento "
"personalisate."

#: pyanaconda/ui/gui/spokes/storage.py:201
#, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:213
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""
"<b>Le spatio pro le installation de %(productName)s es insufficiente</b>, "
"etiam si tu usate tote le spatio disponibile sur le discos seligite. On pote "
"connecter altere discos pro obtener spatio additional, modificar le "
"selection de programma pro installar un variante plus parve de "
"<b>%(productName)s</b>, o quitar le installator."

#: pyanaconda/ui/gui/spokes/storage.py:239
msgctxt "GUI|Spoke"
msgid "Installation _Destination"
msgstr "_Destination del installation"

#: pyanaconda/ui/gui/spokes/storage.py:349
msgid ""
"Partitioning method changed - planned storage configuration changes will be "
"cancelled."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:417
#: pyanaconda/ui/tui/spokes/installation_source.py:136
#: pyanaconda/ui/tui/spokes/storage.py:123
#: pyanaconda/ui/tui/spokes/software_selection.py:159
msgid "Processing..."
msgstr "In processo…"

#: pyanaconda/ui/gui/spokes/storage.py:419
#: pyanaconda/ui/tui/spokes/storage.py:125
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:421
#: pyanaconda/ui/tui/spokes/storage.py:127
msgid "No disks selected"
msgstr "Nulle discos seligite"

#: pyanaconda/ui/gui/spokes/storage.py:423
#: pyanaconda/ui/tui/spokes/storage.py:129
msgid "Error checking storage configuration"
msgstr "Error al verification del configuration del immagazinage"

#: pyanaconda/ui/gui/spokes/storage.py:425
#: pyanaconda/ui/tui/spokes/storage.py:131
msgid "Warning checking storage configuration"
msgstr "Aviso al controlo del configuration de immagazinage"

#: pyanaconda/ui/gui/spokes/storage.py:427
#: pyanaconda/ui/tui/spokes/storage.py:133
msgid "Automatic partitioning selected"
msgstr "Partitionamento automatic seligite"

#: pyanaconda/ui/gui/spokes/storage.py:429
#: pyanaconda/ui/tui/spokes/storage.py:135
msgid "Custom partitioning selected"
msgstr "Partitionamento personalisate seligite"

#: pyanaconda/ui/gui/spokes/storage.py:550
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:554
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:585
#, python-brace-format
msgid ""
"FCP device {hba_id}\n"
"WWPN {wwpn}\n"
"LUN {lun}"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:591
#, python-brace-format
msgid "NVDIMM device {namespace}"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:605
msgid "{} free"
msgstr "{} libere"

#: pyanaconda/ui/gui/spokes/storage.py:707
msgid ""
"You have chosen to skip boot loader installation. Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:885
msgid "There was a problem with your disk selection. Click here for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:990
msgid ""
"The following errors were encountered when checking your disk selection. You "
"can modify your selection or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:995
#: pyanaconda/ui/gui/spokes/storage.py:1016
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/storage.py:996
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Disk Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1011
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""
"Le sequente errores se ha incontrate durante le controlo del configuration "
"de immagazinage. Il pote modificar le schema de immagazinage o quitar le "
"installator."

#: pyanaconda/ui/gui/spokes/storage.py:1017
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr "_Modificar le schema de immagazinage"

#: pyanaconda/ui/gui/spokes/storage.py:1031
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""
"Le sequente avisos se ha incontrate durante le controlo del configuration de "
"immagazinage. Istos non es fatal, ma tu pote vol modificar le schema de "
"immagazinage."

#: pyanaconda/ui/gui/spokes/storage.py:1036
msgctxt "GUI|Storage|Warning Dialog"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/welcome.py:239
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr "BENVENITE A %(name)s %(version)s."

#: pyanaconda/ui/gui/spokes/keyboard.py:50
#, python-format
msgid "%s to switch layouts."
msgstr "%s pro cambiar le dispositiones."

#: pyanaconda/ui/gui/spokes/keyboard.py:279
msgctxt "GUI|Spoke"
msgid "_Keyboard"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:377
msgid "Testing layouts configuration not available."
msgstr "Le proba de dispositiones de clavieros non es disponibile."

#: pyanaconda/ui/gui/spokes/keyboard.py:385
msgid "Getting list of layouts..."
msgstr "Obtenimento del lista de dispositiones..."

#: pyanaconda/ui/gui/spokes/keyboard.py:439
msgid ""
"Keyboard layouts are not supported when using VNC.\n"
"However the settings will be used after the installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:449
msgid "Layout switching not configured."
msgstr "Le cambiamento de disposition non es configurate."

#: pyanaconda/ui/gui/spokes/blivet_gui.py:99
msgctxt "GUI|Spoke"
msgid "_Blivet-GUI Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.py:249
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details</a> or "
"press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.py:251
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details</a> "
"or press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:65
msgid "Setting up installation source..."
msgstr "Configuration del fonte de installation..."

#: pyanaconda/ui/gui/spokes/installation_source.py:108
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:113
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:119
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:124
msgid "Proxy authentication data duplicated"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:266
msgid "This media is good to install from."
msgstr "Iste medio es bon pro installar de illac."

#: pyanaconda/ui/gui/spokes/installation_source.py:268
msgid "This media is not good to install from."
msgstr "Iste medio non es bon pro installar de illac."

#: pyanaconda/ui/gui/spokes/installation_source.py:408
msgctxt "GUI|Spoke"
msgid "_Installation Source"
msgstr "Fonte de _installation"

#: pyanaconda/ui/gui/spokes/installation_source.py:556
msgid "Failed to set up installation source; check the repo url"
msgstr ""
"Impossibile de configurar le fonte de installation; verifica le URL de "
"repositorio"

#: pyanaconda/ui/gui/spokes/installation_source.py:657
#: pyanaconda/ui/gui/spokes/software_selection.py:223
msgid "Checking software dependencies..."
msgstr "Controlo de dependentias de software..."

#: pyanaconda/ui/gui/spokes/installation_source.py:661
msgid "Error setting up base repository"
msgstr "Error al configuration del repositorio fundamental"

#: pyanaconda/ui/gui/spokes/installation_source.py:663
#: pyanaconda/ui/tui/spokes/installation_source.py:120
#: pyanaconda/ui/tui/spokes/installation_source.py:134
msgid "Error setting up software source"
msgstr "Error al configurar un fonte de software"

#: pyanaconda/ui/gui/spokes/installation_source.py:667
#: pyanaconda/ui/tui/spokes/installation_source.py:113
#, python-format
msgid "NFS server %s"
msgstr "Servitor NFS %s"

#: pyanaconda/ui/gui/spokes/installation_source.py:669
#: pyanaconda/ui/tui/spokes/installation_source.py:115
msgid "Local media"
msgstr "Medio local"

#: pyanaconda/ui/gui/spokes/installation_source.py:671
#: pyanaconda/ui/tui/spokes/installation_source.py:117
msgid "Local media via SE/HMC"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:677
msgid "Error setting up installation from HDD"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:680
#: pyanaconda/ui/tui/spokes/installation_source.py:123
#: pyanaconda/ui/tui/spokes/installation_source.py:166
#: pyanaconda/ui/gui/spokes/installation_source.glade:775
msgid "Closest mirror"
msgstr "Speculo le plus vicin"

#: pyanaconda/ui/gui/spokes/installation_source.py:829
msgid "Failed to set up installation source; check the repo url."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:831
msgid ""
"Failed to set up installation source; check the repo url and proxy settings."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:901
#, python-format
msgid "Device: %s"
msgstr "Dispositivo: %s"

#: pyanaconda/ui/gui/spokes/installation_source.py:902
#, python-format
msgid "Label: %s"
msgstr "Etiquetta: %s"

#: pyanaconda/ui/gui/spokes/installation_source.py:1046
#: pyanaconda/ui/tui/spokes/installation_source.py:159
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1054
msgid "You need to configure the network to use a network installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1144
#, python-format
msgid "Repository %s has empty url"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1146
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1151
#, python-format
msgid "Repository %s has invalid url"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1153
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1160
#, python-format
msgid "Repository %s does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1162
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1166
#, python-format
msgid "Repository %s has empty NFS server"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1168
msgid "NFS server is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1176
#, python-format
msgid "Repository %s has invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1178
msgid "Invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1182
#, python-format
msgid "Repository %s required remote directory"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1184
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1205
msgid "Duplicate repository names."
msgstr "Duplicitate de nomines de repositorios."

#: pyanaconda/ui/gui/spokes/installation_source.py:1213
msgid "Empty repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1216
msgid "Invalid repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.py:1221
msgid "Repository name conflicts with internal repository name."
msgstr ""
"Le nomine de repositorio conflige con le nomine de un repositorio interne."

#: pyanaconda/ui/gui/spokes/software_selection.py:64
msgctxt "GUI|Spoke"
msgid "_Software Selection"
msgstr "_Selection de programmas"

#: pyanaconda/ui/gui/spokes/software_selection.py:228
msgid "Error checking software dependencies"
msgstr "Error al controlo de dependentias de software"

#: pyanaconda/ui/gui/spokes/software_selection.py:297
#: pyanaconda/ui/tui/spokes/software_selection.py:157
msgid "Error checking software selection"
msgstr "Error al verification del selection del programmas"

#: pyanaconda/ui/gui/spokes/software_selection.py:300
#: pyanaconda/ui/tui/spokes/software_selection.py:161
msgid "Installation source not set up"
msgstr "Le fonte de installation non es configurate"

#: pyanaconda/ui/gui/spokes/software_selection.py:303
#: pyanaconda/ui/tui/spokes/software_selection.py:163
msgid "Source changed - please verify"
msgstr "Fonte cambiate - verifica"

#: pyanaconda/ui/gui/spokes/software_selection.py:310
#: pyanaconda/ui/tui/spokes/software_selection.py:170
msgid "Custom software selected"
msgstr "Programma personalisate seligite"

#: pyanaconda/ui/gui/spokes/software_selection.py:313
msgid "Invalid environment specified in kickstart"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:326
msgid "Selected environment is not valid"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:494
msgid ""
"Error checking software dependencies.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:609
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""

#: pyanaconda/ui/gui/spokes/software_selection.py:615
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/software_selection.py:616
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr "_Modificar le fonte de programmas"

#: pyanaconda/ui/gui/spokes/software_selection.py:617
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr "Modificar _selectiones"

#: pyanaconda/ui/gui/spokes/language_support.py:63
msgctxt "GUI|Spoke"
msgid "_Language Support"
msgstr ""

#: pyanaconda/ui/lib/storage.py:205
#, python-brace-format
msgid "{count} disk selected; {capacity} capacity; {free} free"
msgid_plural "{count} disks selected; {capacity} capacity; {free} free"
msgstr[0] ""

#: pyanaconda/ui/lib/storage.py:271
msgid "Saving storage configuration..."
msgstr "Salveguarda del configuration de immagazinage..."

#: pyanaconda/ui/lib/storage.py:276
msgid "Failed to save storage configuration"
msgstr "Impossibile salvar le configuration de immagazinage"

#: pyanaconda/ui/lib/storage.py:281
msgid "Failed to save boot loader configuration"
msgstr "Impossibile salvar le configuration del cargator de initio"

#: pyanaconda/ui/lib/storage.py:285
msgid "Checking storage configuration..."
msgstr "Verification del configuration de immagazinage..."

#: pyanaconda/ui/lib/format_dasd.py:107
msgid "Nothing to format"
msgstr ""

#: pyanaconda/ui/lib/format_dasd.py:111
msgid "Formatting DASDs"
msgstr "Formatation de DASD"

#: pyanaconda/ui/lib/format_dasd.py:115
msgid "Probing storage"
msgstr "Probante immagazinage"

#: pyanaconda/ui/lib/space.py:103
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional {} is needed."
msgstr ""

#: pyanaconda/ui/lib/space.py:106
msgid "Not enough space in file systems for the current software selection."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:57
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:54
msgid "Default HUB title"
msgstr "Titulo de central predefinite"

#: pyanaconda/ui/tui/hubs/__init__.py:137 pyanaconda/ui/tui/hubs/summary.py:150
msgid "Please complete all spokes before continuing"
msgstr "Comple tote le succursales ante de continuar"

#: pyanaconda/ui/tui/hubs/__init__.py:162
#, python-format
msgid "to enter the %(spoke_title)s spoke"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:47
msgid "Installation"
msgstr "Installation"

#: pyanaconda/ui/tui/hubs/summary.py:61
msgid "Starting automated install"
msgstr "Initio de installation automatic"

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:106
#, python-format
msgid "Enter '%s' to ignore the warning and attempt to install anyway."
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:108 pyanaconda/ui/tui/hubs/summary.py:135
#: pyanaconda/ui/tui/hubs/summary.py:147
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:117
msgid "The following mandatory spokes are not completed:"
msgstr "Le sequente succursales obligatori non es complite:"

#: pyanaconda/ui/tui/hubs/summary.py:135
msgid "to begin installation"
msgstr ""

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:160
#: pyanaconda/ui/tui/spokes/installation_source.py:498
#: pyanaconda/ui/tui/spokes/network.py:565
#: pyanaconda/ui/tui/spokes/storage.py:254
#: pyanaconda/ui/tui/spokes/storage.py:512
#: pyanaconda/ui/tui/spokes/storage.py:584
#: pyanaconda/ui/tui/spokes/storage.py:653
#: pyanaconda/ui/tui/spokes/software_selection.py:252
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr "c"

#: pyanaconda/ui/tui/spokes/unsupported_hardware.py:38
#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:68
msgid "Unsupported Hardware Detected"
msgstr "Hardware non supportate detegite"

#: pyanaconda/ui/tui/spokes/installation_source.py:68
msgid "Installation source"
msgstr "Fonte de installation"

#: pyanaconda/ui/tui/spokes/installation_source.py:180
msgid "Choose an installation source type."
msgstr "Selige un typo del fonte de installation."

#: pyanaconda/ui/tui/spokes/installation_source.py:181
msgid "CD/DVD"
msgstr "CD/DVD"

#: pyanaconda/ui/tui/spokes/installation_source.py:182
msgid "local ISO file"
msgstr "file ISO local"

#: pyanaconda/ui/tui/spokes/installation_source.py:183
msgid "Network"
msgstr "Rete"

#: pyanaconda/ui/tui/spokes/installation_source.py:186
msgid "SE/HMC"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:266
#: pyanaconda/ui/tui/spokes/installation_source.py:319
msgid "Specify Repo Options"
msgstr "Specificar optiones de repositorio"

#: pyanaconda/ui/tui/spokes/installation_source.py:278
msgid "Repo URL"
msgstr "URL de repositorio"

#: pyanaconda/ui/tui/spokes/installation_source.py:338
msgid "SERVER:/PATH"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:342
msgid "NFS mount options"
msgstr "Optiones de montage NFS"

#: pyanaconda/ui/tui/spokes/installation_source.py:352
msgid "Server must be specified as SERVER:/PATH"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_source.py:396
msgid "Select device containing the ISO file"
msgstr "Seliger un dispositivo continente le file ISO"

#: pyanaconda/ui/tui/spokes/installation_source.py:439
msgid "No mountable devices found"
msgstr "Nulle dispositivos trovate que es appropriate pro montage"

#: pyanaconda/ui/tui/spokes/installation_source.py:468
msgid "Select an ISO to use as install source"
msgstr "Seliger un ISO pro usar como un fonte de installation"

#: pyanaconda/ui/tui/spokes/installation_source.py:486
msgid "No *.iso files found in device root folder"
msgstr ""
"Nulle files *.iso es trovate in le directorio de radice del dispositivo"

#: pyanaconda/ui/tui/spokes/__init__.py:57
msgid "Default spoke title"
msgstr "Titulo de succursal predefinite"

#: pyanaconda/ui/tui/spokes/__init__.py:61
msgid "testing status..."
msgstr "verification del stato..."

#: pyanaconda/ui/tui/spokes/askvnc.py:49
msgid "VNC"
msgstr "VNC"

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/spokes/askvnc.py:102
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/askvnc.py:126
msgid "VNC Password"
msgstr "Contrasigno VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:131
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""
"Forni un contrasigno VNC (illo debe esser de 6 a 8 characteres).\n"
"Tu debera facer lo duo vices. Lassa lo in blanco pro le uso sin contrasigno"

#: pyanaconda/ui/tui/spokes/askvnc.py:149
msgid "Password (confirm): "
msgstr "Contrasigno (confirma): "

#: pyanaconda/ui/tui/spokes/askvnc.py:152
msgid "Passwords do not match!"
msgstr "Le contrasignos non correspondente!"

#: pyanaconda/ui/tui/spokes/askvnc.py:154
msgid "The password must be at least six characters long."
msgstr "Le contrasigno debe haber al minus sex characteres de longitude."

#: pyanaconda/ui/tui/spokes/askvnc.py:157
msgid "The password cannot be more than eight characters long."
msgstr "Le contrasigno non pote esser plus longe que 8 characteres."

#: pyanaconda/ui/tui/spokes/language_support.py:51
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:85
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:97
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:102
msgid "Available languages"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/language_support.py:126
#: pyanaconda/ui/tui/spokes/language_support.py:137
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:135
msgid "Please select language support to install"
msgstr ""

#: pyanaconda/ui/tui/spokes/language_support.py:137
msgid "to return to language list"
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:43
msgid "Root password"
msgstr "Contrasigno del superusator"

#: pyanaconda/ui/tui/spokes/root_password.py:75
msgid "Disabled. Set password to enable root account."
msgstr ""

#: pyanaconda/ui/tui/spokes/root_password.py:80
msgid "Password is set."
msgstr "Le contrasigno es definite."

#: pyanaconda/ui/tui/spokes/root_password.py:82
msgid "Password is not set."
msgstr "Le contrasigno non es definite."

#: pyanaconda/ui/tui/spokes/root_password.py:87
msgid "Please select new root password. You will have to type it twice."
msgstr ""
"Selige un nove contrasigno de superusator. Tu debera introducer lo duo vices."

#: pyanaconda/ui/tui/spokes/root_password.py:93
#: pyanaconda/ui/tui/spokes/user.py:140 pyanaconda/ui/gui/spokes/user.glade:172
#: pyanaconda/ui/gui/spokes/root_password.glade:87
msgid "Password"
msgstr "Contrasigno"

#: pyanaconda/ui/tui/spokes/network.py:51
msgid "Bad format of the IP address"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:52
msgid "Bad format of the netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:207
msgid "Network configuration"
msgstr "Configuration de rete"

#: pyanaconda/ui/tui/spokes/network.py:268
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr "Interfacie con filo (%(interface_name)s) es disconnectite\n"

#: pyanaconda/ui/tui/spokes/network.py:273
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr "Interfacie con filo (%(interface_name)s) es connectite\n"

#: pyanaconda/ui/tui/spokes/network.py:289
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""
" Adresse IPv4: %(addr)s Masca de rete: %(netmask)s Passarella: %(gateway)s\n"

#: pyanaconda/ui/tui/spokes/network.py:291
#, python-format
msgid " DNS: %s\n"
msgstr " DNS: %s\n"

#: pyanaconda/ui/tui/spokes/network.py:300
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr " Adresse IPv6: %(addr)s/%(prefix)d\n"

#: pyanaconda/ui/tui/spokes/network.py:311
msgid "Network configuration is not available."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:317
#, python-format
msgid "Host Name: %s\n"
msgstr "Nomine de Hospite: %s\n"

#: pyanaconda/ui/tui/spokes/network.py:319
#, python-format
msgid "Current host name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:326
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:239
#: pyanaconda/ui/gui/spokes/network.glade:2210
msgid "Host Name"
msgstr "Nomine de hospite"

#: pyanaconda/ui/tui/spokes/network.py:327
msgid "Set host name"
msgstr "Fixa nomine de hospite"

#: pyanaconda/ui/tui/spokes/network.py:331
#, python-format
msgid "Configure device %s"
msgstr "Configurar le dispositivo %s"

#: pyanaconda/ui/tui/spokes/network.py:441
msgid "Device configuration"
msgstr "Configuration de dispositivo"

#: pyanaconda/ui/tui/spokes/network.py:463
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr "Adresse IPv4 o %s pro DHCP"

#: pyanaconda/ui/tui/spokes/network.py:467
msgid "IPv4 netmask"
msgstr "Masca de rete IPv4"

#: pyanaconda/ui/tui/spokes/network.py:470
msgid "IPv4 gateway"
msgstr "Passarella IPv4"

#: pyanaconda/ui/tui/spokes/network.py:473
#, python-format
msgid ""
"IPv6 address[/prefix] or %(auto)s for automatic, %(dhcp)s for DHCP, "
"%(ignore)s to turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:479
msgid "IPv6 default gateway"
msgstr "Passarella predefinite IPv6"

#: pyanaconda/ui/tui/spokes/network.py:482
msgid "Nameservers (comma separated)"
msgstr "Servitores de nomines (separate per commas)"

#: pyanaconda/ui/tui/spokes/network.py:485
msgid "Connect automatically after reboot"
msgstr "Connecter automaticamente post reinitio"

#: pyanaconda/ui/tui/spokes/network.py:489
msgid "Apply configuration in installer"
msgstr "Applicar le configuration in le installator"

#: pyanaconda/ui/tui/spokes/network.py:495
#, python-format
msgid "Configuring device %s."
msgstr "Configuration del dispositivo %s."

#: pyanaconda/ui/tui/spokes/network.py:567
msgid "Configuration not saved: netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:47
msgid "status: working"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:48
msgid "status: not working"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:49
msgid "checking status"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:66
msgid "Time settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:204
msgid "Timezone is not set."
msgstr "Fuso horari non es definite."

#: pyanaconda/ui/tui/spokes/time_spoke.py:215
msgid "not set"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:219
#, python-format
msgid "Timezone: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:225
#: pyanaconda/ui/tui/spokes/time_spoke.py:390
msgid "NTP servers:"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:230
msgid "not configured"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:241
msgid "Change timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:243
msgid "Set timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:248
msgid "Configure NTP servers"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:284
msgid "Timezone settings"
msgstr "Parametros de fuso horari"

#: pyanaconda/ui/tui/spokes/time_spoke.py:311
#, python-format
msgid "Available timezones in region %s"
msgstr "Fusos horari disponibile in le region %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:315
msgid "Available regions"
msgstr "Regiones disponibile"

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/time_spoke.py:356
#: pyanaconda/ui/tui/spokes/time_spoke.py:367
msgctxt "TUI|Spoke Navigation|Time Settings"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:365
msgid "Please select the timezone. Use numbers or type names directly"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:367
msgid "back to region list"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:380
msgid "NTP configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:395
msgid "no NTP servers have been configured"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:406
msgid "Add NTP server"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:410
msgid "Remove NTP server"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:439
msgid "Add NTP server address"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:455
#, python-format
msgid "Enter an NTP server address and press %s"
msgstr "Insere un adresse de servitor NTP e pressa sur %s"

#: pyanaconda/ui/tui/spokes/time_spoke.py:479
msgid "Select an NTP server to remove"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:61
msgid "Use All Space"
msgstr "Usar tote le spatio"

#: pyanaconda/ui/tui/spokes/storage.py:62
msgid "Replace Existing Linux system(s)"
msgstr "Reimplaciar le systema(s) Linux existente"

#: pyanaconda/ui/tui/spokes/storage.py:63
msgid "Use Free Space"
msgstr "Usar le spatio libere"

#: pyanaconda/ui/tui/spokes/storage.py:81
msgid "Installation Destination"
msgstr "Destination de installation"

#: pyanaconda/ui/tui/spokes/storage.py:201
msgid "Select all"
msgstr "Seliger toto"

#: pyanaconda/ui/tui/spokes/storage.py:305
msgid ""
"The following unformatted or LDL DASDs have been detected on your system. "
"You can choose to format them now with dasdfmt or cancel to leave them "
"unformatted. Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:312
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:321
msgid "This may take a moment."
msgstr "Isto pote prender un momento."

#: pyanaconda/ui/tui/spokes/storage.py:337
msgid "Passphrase"
msgstr "Phrase de contrasigno"

#: pyanaconda/ui/tui/spokes/storage.py:338
msgid ""
"Please provide a default LUKS passphrase for all devices you want to "
"encrypt. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:416
msgid "Partitioning Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:453
msgid "Manually assign mount points"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:460
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target or manually assign mount points."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:505
msgid "Reverting previous configuration. This may take a moment..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:537
msgid "Partition Scheme Options"
msgstr "Optiones del schema de partitiones"

#: pyanaconda/ui/tui/spokes/storage.py:573
msgid "Select a partition scheme configuration."
msgstr "Selige un configuration del schema de partitiones."

#: pyanaconda/ui/tui/spokes/storage.py:605
msgid "Assign mount points"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:626
msgid ""
"Choose device from above to assign mount point and set format.\n"
"Formats marked with * are new formats meaning ALL DATA on the original "
"format WILL BE LOST!"
msgstr ""

#. TRANSLATORS: 's' to rescan devices
#: pyanaconda/ui/tui/spokes/storage.py:638
#: pyanaconda/ui/tui/spokes/storage.py:648
msgctxt "TUI|Spoke Navigation|Partitioning"
msgid "s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:638
msgid "rescan devices"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:710
msgid ""
"Warning: This will revert all changes done so far.\n"
"Do you want to proceed?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:719
msgid "Scanning disks. This may take a moment..."
msgstr "Scannation de discos. Isto pote prender un momento…"

#: pyanaconda/ui/tui/spokes/storage.py:733
#, python-format
msgid "Configure device: %s"
msgstr "Configurar le dispositivo: %s"

#: pyanaconda/ui/tui/spokes/storage.py:753
msgid "Choose from above to assign mount point and/or set format."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:769
#: pyanaconda/ui/gui/spokes/lib/summary.glade:177
msgid "Mount point"
msgstr "Puncto de montage"

#: pyanaconda/ui/tui/spokes/storage.py:776
#: pyanaconda/ui/tui/spokes/storage.py:781
#: pyanaconda/ui/tui/spokes/storage.py:813
msgid "none"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:799
msgid "Invalid mount point given"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:812
msgid "Format"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:822
msgid "Invalid or unsupported format given"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:824
#, python-format
msgid "Supported formats: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:841
msgid "Reformat"
msgstr "Reformatar"

#: pyanaconda/ui/tui/spokes/shell_spoke.py:41
msgid "Shell"
msgstr "Shell"

#: pyanaconda/ui/tui/spokes/shell_spoke.py:55
msgid "Start shell"
msgstr "Initia shell"

#: pyanaconda/ui/tui/spokes/shell_spoke.py:63
msgid "Exit the shell to continue"
msgstr "Exi le shell pro continuar"

#: pyanaconda/ui/tui/spokes/user.py:39
msgid "Full name can't contain the ':' character"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:73
msgid "User creation"
msgstr "Creation de usator"

#: pyanaconda/ui/tui/spokes/user.py:126
msgid "Create user"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:130
msgid "Full name"
msgstr "Nomine complete"

#: pyanaconda/ui/tui/spokes/user.py:133
msgid "User name"
msgstr "Nomine de usator"

#: pyanaconda/ui/tui/spokes/user.py:136
msgid "Use password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:148
msgid "Administrator"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:152
msgid "Groups"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:229
msgid "You must set a password"
msgstr "Tu debe fixar un contrasigno"

#: pyanaconda/ui/tui/spokes/user.py:247
#, python-format
msgid ""
"Invalid user name: %(name)s.\n"
"%(error_message)s"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:54
msgid "Software selection"
msgstr "Selection de software"

#: pyanaconda/ui/tui/spokes/software_selection.py:198
msgid "Installation source needs to be set up first."
msgstr "Le fonte de installation debe esser configurate in prime loco."

#: pyanaconda/ui/tui/spokes/software_selection.py:222
msgid "Base environment"
msgstr "Ambiente basic"

#: pyanaconda/ui/tui/spokes/software_selection.py:232
msgid "Add-ons for selected environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/software_selection.py:234
msgid "No add-ons to select."
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:51
msgid "Progress"
msgstr "Progresso"

#: pyanaconda/ui/tui/spokes/installation_progress.py:134
msgid "Installation complete"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:136
msgid "Use of this product is subject to the license agreement found at:"
msgstr ""

#: pyanaconda/ui/tui/spokes/installation_progress.py:147
#, python-format
msgid "Installation complete. Press %s to quit"
msgstr "Installation completate. Pressa sur %s pro quitar"

#: pyanaconda/ui/tui/tuiobject.py:142
#, python-format
msgid "Enter a new value for '%(title)s' and press %(enter)s"
msgstr "Insere un nove valor pro '%(title)s' e pressa sur %(enter)s"

#: pyanaconda/ui/tui/tuiobject.py:271
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""
"\n"
"Vole tu usar lo comocunque?"

#: pyanaconda/bootloader/grub2.py:487
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""

#: pyanaconda/bootloader/grub2.py:526
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader/grub2.py:533
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader/base.py:149
msgid "/boot file system"
msgstr ""

#: pyanaconda/bootloader/base.py:298
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""
"Le insimules de RAID que contine '%(desc)s' debe haber un del sequente "
"nivellos de RAID: %(raid_level)s."

#: pyanaconda/bootloader/base.py:306
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""
"Le insimules de RAID que contine '%(desc)s' debe haber un del sequente "
"versiones de metadatos: %(metadata_versions)s."

#: pyanaconda/bootloader/base.py:314
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""
"Le insimules de RAID que contine '%(desc)s' debe haber un del sequente typos "
"de dispositivo: %(types)s."

#: pyanaconda/bootloader/base.py:330
#, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr ""
"%(name)s debe haber un del sequente typos de etiquetta de disco: %(types)s."

#: pyanaconda/bootloader/base.py:342
#, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr "%(desc)s non pote esser del typo %(type)s."

#: pyanaconda/bootloader/base.py:348
#, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr "%(desc)s debe esser montate sur un de %(mountpoints)s."

#: pyanaconda/bootloader/base.py:360
#, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr "Le dimension de %(desc)s debe esser inter %(min)d e %(max)d MB"

#: pyanaconda/bootloader/base.py:366
#, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr "%(desc)s non debe esser plus parve que %(min)d MB."

#: pyanaconda/bootloader/base.py:375
#, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr "%(desc)s non debe esser plus grande que %(max)d MB."

#: pyanaconda/bootloader/base.py:393
#, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr "%(desc)s debe esser intra le prime %(max_end)s del disco."

#: pyanaconda/bootloader/base.py:403
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s debe esser sur un partition primari."

#: pyanaconda/bootloader/base.py:488
msgid ""
"Boot loader stage1 device cannot be on an iSCSI disk which is not configured "
"in iBFT."
msgstr ""

#: pyanaconda/bootloader/base.py:545 pyanaconda/bootloader/base.py:659
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s non pote esser sur un dispositivo de blocos cryptate."

#: pyanaconda/bootloader/base.py:611
#, python-format
msgid ""
"%(bootloader_stage2_description)s cannot be on an iSCSI disk which is not "
"configured in iBFT."
msgstr ""

#: pyanaconda/bootloader/base.py:618
#, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr "%(desc)s non pote esser del typo %(type)s"

#: pyanaconda/bootloader/execution.py:175
msgid "No match found for given boot drive \"{}\"."
msgstr ""

#: pyanaconda/bootloader/execution.py:179
msgid "More than one match found for given boot drive \"{}\"."
msgstr ""

#: pyanaconda/bootloader/execution.py:183
msgid "Requested boot drive \"{}\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/input_checking.py:539
msgid "Full name cannot contain colon characters"
msgstr "Le nomine integre non pote continer characteres de duo punctos"

#: pyanaconda/network.py:110
msgid "Host name cannot be None or an empty string."
msgstr "Nomine de hospite non pote esser None o un catena vacue."

#: pyanaconda/network.py:113
msgid "Host name must be 255 or fewer characters in length."
msgstr ""

#: pyanaconda/network.py:116
msgid "Local host name must not end with period '.'."
msgstr ""

#: pyanaconda/network.py:119
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:465
msgid "Status not available"
msgstr ""

#: pyanaconda/network.py:470
msgid "Connecting..."
msgstr "Connectente…"

#: pyanaconda/network.py:472
msgid "Disconnecting..."
msgstr "Disconnectente..."

#: pyanaconda/network.py:503
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr "Interfacie con filo (%(interface_name)s) es connectite"

#: pyanaconda/network.py:506
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr "Interfacie sin filo es connectite a %(access_point)s"

#: pyanaconda/network.py:509
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""
"Interfacie aggregate %(interface_name)s (%(list_of_slaves)s) es connectite"

#: pyanaconda/network.py:513
#, python-format
msgid "Team %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:517
#, python-format
msgid "Bridge %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:523
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:542
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr "Connectite: %(list_of_interface_names)s"

#: pyanaconda/network.py:544
msgid "Not connected"
msgstr "Non connectite"

#: pyanaconda/network.py:547 pyanaconda/ui/gui/spokes/network.glade:2061
msgid "No network devices available"
msgstr "Nulle dispositivos de rete disponibile"

#: pyanaconda/storage/fsset.py:390
#, python-brace-format
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type. The file says that {detected_type} at "
"{mount_point} is {fstab_type}."
msgstr ""

#: pyanaconda/storage/root.py:130
#, python-format
msgid "Linux on %s"
msgstr ""

#: pyanaconda/storage/root.py:135
msgid "Unknown Linux"
msgstr "Linux incognite"

#: pyanaconda/storage/root.py:137
#, python-format
msgid "%(product)s %(version)s for %(arch)s"
msgstr ""

#: pyanaconda/storage/root.py:140
#, python-format
msgid "%(product)s Linux %(version)s for %(arch)s"
msgstr ""

#: pyanaconda/storage/utils.py:63
msgid "Unsupported"
msgstr ""

#: pyanaconda/storage/utils.py:64 pyanaconda/storage/utils.py:89
msgid "LVM"
msgstr "LVM"

#: pyanaconda/storage/utils.py:65
msgid "RAID"
msgstr "RAID"

#: pyanaconda/storage/utils.py:66 pyanaconda/storage/utils.py:87
msgid "Standard Partition"
msgstr "Partition normal"

#: pyanaconda/storage/utils.py:67 pyanaconda/storage/utils.py:88
msgid "Btrfs"
msgstr ""

#: pyanaconda/storage/utils.py:68 pyanaconda/storage/utils.py:90
msgid "LVM Thin Provisioning"
msgstr "LVM con allocation fin e dynamic"

#: pyanaconda/storage/utils.py:69 pyanaconda/ui/gui/spokes/lib/resize.glade:150
msgid "Disk"
msgstr "Disco"

#: pyanaconda/storage/utils.py:74
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""
"Le area «swap» del computator es usate per le systema\n"
"operative quando illo ha carentia de memoria."

#: pyanaconda/storage/utils.py:76
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""
"Le area «boot» del computator es un loco ubi reside \n"
"files necessari pro initiar le systema operative."

#: pyanaconda/storage/utils.py:78
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""
"Le area «root» del computator es un loco ubi reside\n"
"files principal e applicationes del systema."

#: pyanaconda/storage/utils.py:80
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""
"Le area «home» del computator es un loco ubi reside tote\n"
"le datos personal de usatores."

#: pyanaconda/storage/utils.py:82
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""
"Le partition «BIOS boot» se require pro permitter le initio \n"
"de discos partitionate GPT sur apparato con BIOS."

#: pyanaconda/storage/utils.py:84
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/storage/utils.py:398
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr ""
"Le certificato de deposito fiduciari %s necessita un connexion al rete."

#: pyanaconda/storage/utils.py:406
#, python-format
msgid ""
"SSL error while downloading the escrow certificate:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/storage/utils.py:408
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Le sequente error ha essite incontrate durante le discargamento del "
"certificato de deposito fiduciari:\n"
"\n"
"%s"

#: pyanaconda/storage/utils.py:495
#, python-format
msgid ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disk %(unselected)s. You must select or de-select these disks as "
"a set."
msgid_plural ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disks %(unselected)s. You must select or de-select these disks as "
"a set."
msgstr[0] ""

#: pyanaconda/storage/checker.py:51
#, python-format
msgid ""
"Your root partition is less than %(size)s which is usually too small to "
"install %(product)s."
msgstr ""

#: pyanaconda/storage/checker.py:57
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: pyanaconda/storage/checker.py:62
#: pyanaconda/modules/storage/partitioning/interactive/utils.py:437
msgid "You must create a new file system on the root device."
msgstr ""

#: pyanaconda/storage/checker.py:88
msgid ""
"This platform requires /boot on a dedicated partition or logical volume. If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""

#: pyanaconda/storage/checker.py:95
#, python-brace-format
msgid ""
"The LDL DASD disk {name} ({busid}) cannot be used for the installation. "
"Please format it."
msgstr ""

#: pyanaconda/storage/checker.py:117
#, python-format
msgid ""
"It is recommended to create a new file system on your %(mount)s partition."
msgstr ""

#: pyanaconda/storage/checker.py:133
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage/checker.py:155
#, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage/checker.py:161
#, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage/checker.py:179
msgid "No valid boot loader target device found. See below for details."
msgstr ""

#: pyanaconda/storage/checker.py:194
msgid "You have not created a bootable partition."
msgstr ""

#: pyanaconda/storage/checker.py:229
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""

#: pyanaconda/storage/checker.py:251
#, python-format
msgid ""
"You have not specified a swap partition. %(requiredMem)s of memory is "
"recommended to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage/checker.py:258
#, python-format
msgid ""
"You have not specified a swap partition. %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage/checker.py:264
msgid ""
"You have not specified a swap partition. Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage/checker.py:282
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""
"Al minus un del dispositivos de swap non ha un UUID, que es commun quando le "
"spatio de swap ha essite create per versiones plus vetere de mkswap. Iste "
"dispositivos essera indicate per lor percursos in le file /etc/fstab, que "
"non es ideal proque le percursos de dispositivos pote cambiar se secundo le "
"circumstantias. "

#: pyanaconda/storage/checker.py:301
#, python-format
msgid ""
"This mount point is invalid. The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage/checker.py:318
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "Le puncto de montage %s debe esser sur un systema de files de linux."

#: pyanaconda/storage/checker.py:342
msgid ""
"The existing unlocked LUKS device {} cannot be used for the installation "
"without an encryption key specified for this device. Please, rescan the "
"storage."
msgstr ""

#: pyanaconda/storage/checker.py:363
#, python-format
msgid ""
"Encryption requested for LUKS device %s but no encryption key specified for "
"this device."
msgstr ""

#: pyanaconda/storage/checker.py:385
#, python-format
msgid ""
"The available memory is less than %(size)s which can be too small for LUKS2 "
"format. It may fail."
msgstr ""

#: pyanaconda/storage/checker.py:412
#, python-format
msgid ""
"%s is currently mounted and cannot be used for the installation. Please "
"unmount it and retry."
msgstr ""

#: pyanaconda/storage/checker.py:447
msgid ""
"Selected disks {} contain volume group '{}' that also uses further "
"unselected disks. You must select or de-select all these disks as a set."
msgstr ""

#: pyanaconda/startup_utils.py:79
#, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""
"%(product_name)s require %(needed_ram)s MB de memoria pro le installation, "
"ma iste machina ha solmente %(total_ram)s MB.\n"

#: pyanaconda/startup_utils.py:81
#, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""

#: pyanaconda/startup_utils.py:84
msgid ""
"\n"
"Press [Enter] to reboot your system.\n"
msgstr ""
"\n"
"Pressa sur [Enter] pro reinitiar le systema.\n"

#: pyanaconda/startup_utils.py:86
msgid "Not enough RAM"
msgstr "RAM insufficiente"

#: pyanaconda/startup_utils.py:87
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
" Essaya le installator in modo textual con le commando:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" entrate in un terminal de superusator."

#: pyanaconda/startup_utils.py:89
msgid " Starting text mode."
msgstr " Initio del modo textual."

#: pyanaconda/startup_utils.py:126
msgid "The installation cannot continue and the system will be rebooted"
msgstr ""

#: pyanaconda/startup_utils.py:127
msgid "Press ENTER to continue"
msgstr "Pressa sur INTRO pro continuar"

#: pyanaconda/startup_utils.py:224
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr "Exeque ssh install@%s pro comenciar le installation."

#: pyanaconda/startup_utils.py:226
msgid "Please ssh install@HOSTNAME to continue installation."
msgstr "Executa ssh install@HOSTNAME pro continuar le installation."

#: pyanaconda/payload/manager.py:75
msgid "Failed to set up installation source"
msgstr "Impossibile de configurar le fonte de installation"

#: pyanaconda/payload/manager.py:76
msgid "Error downloading package metadata"
msgstr "Error al discargamento del metadatos de pacchetto"

#: pyanaconda/payload/flatpak.py:205
#, python-format
msgid "Installing %(flatpak_name)s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:127
#, python-format
msgid "Receiving objects: %(percent)d%% (%(fetched)d/%(requested)d) %(bytes)s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:133
msgid "Writing objects"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:232
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:262
#, python-format
msgid "Preparing deployment of %s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:282
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:285
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:504
msgid "Starting Flatpak installation"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:521
msgid "Post-installation flatpak tasks"
msgstr ""

#: pyanaconda/payload/rpmostreepayload.py:527
msgid "Flatpak installation has finished"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:246
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1017
msgid "Starting package installation process"
msgstr "Initio del processo de installation de pacchettos"

#: pyanaconda/payload/dnfpayload.py:1038
msgid "Downloading packages"
msgstr "Discargamento de pacchettos"

#: pyanaconda/payload/dnfpayload.py:1051
msgid "Preparing transaction from installation source"
msgstr "Preparation de transaction ab le fonte de installation"

#: pyanaconda/payload/dnfpayload.py:1064
#, python-format
msgid "Installing %s"
msgstr "Installation de %s"

#: pyanaconda/payload/dnfpayload.py:1067
#, python-format
msgid "Configuring %s"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1070
#, python-format
msgid "Verifying %s"
msgstr ""

#: pyanaconda/payload/dnfpayload.py:1075 pyanaconda/payload/livepayload.py:205
#: pyanaconda/installation.py:337
msgid "Performing post-installation setup tasks"
msgstr "Execution de cargas de configuration post installation"

#: pyanaconda/payload/livepayload.py:104 pyanaconda/payload/livepayload.py:126
msgid "Installing software"
msgstr "Installation de software"

#: pyanaconda/payload/livepayload.py:279 pyanaconda/payload/livepayload.py:288
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr "Discargamento de %(url)s (%(pct)d%%)"

#: pyanaconda/payload/livepayload.py:440
msgid "Checking image checksum"
msgstr "Verification del summa de controlo del imagine"

#: pyanaconda/exception.py:109
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"Le installation se ha stoppate a causa de un problema possibile con le "
"apparato. Le message exacte del error es:\n"
"\n"
"%s.\n"
"\n"
"Nunc le installator se terminara."

#: pyanaconda/exception.py:113
msgid "Hardware error occurred"
msgstr ""

#: pyanaconda/exception.py:170
#, python-format
msgid ""
"\n"
"The installation was stopped due to an error which occurred while running in "
"non-interactive cmdline mode. Since there cannot be any questions in cmdline "
"mode, edit your kickstart file and retry installation. \n"
"The exact error message is: \n"
"\n"
"%s. \n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:178
#, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/vnc.py:79
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "Installation de %(productName)s %(productVersion)s"

#: pyanaconda/vnc.py:148
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""
"Installation de %(productName)s %(productVersion)s sur le host %(name)s"

#: pyanaconda/vnc.py:167
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Tentativa de connecter al cliente VNC sur le host %s..."

#: pyanaconda/vnc.py:181
msgid "Connected!"
msgstr "Connectite!"

#: pyanaconda/vnc.py:184
msgid "Will try to connect again in 15 seconds..."
msgstr "Reprobara de connecter in 15 secundas..."

#: pyanaconda/vnc.py:191
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "Abandono del connexion post %d tentativa!\n"
msgstr[1] "Abandono del connexion post %d tentativas!\n"

#: pyanaconda/vnc.py:199
msgid "Attempting to start vncconfig"
msgstr ""

#: pyanaconda/vnc.py:212
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"Connecte manualmente le cliente VNC a %s pro comenciar le installation."

#: pyanaconda/vnc.py:214
#, python-format
msgid ""
"Please manually connect your vnc client to IP-ADDRESS:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the IP-"
"ADDRESS."
msgstr ""
"Connecte manualmente le cliente VNC a ADRESSE-IP:%s pro comenciar le "
"installation. Aperi le shell (Ctrl-B 2) e executar «ip addr» pro trovar le "
"ADRESSE-IP."

#: pyanaconda/vnc.py:219
msgid "Starting VNC..."
msgstr "Initio de VNC..."

#: pyanaconda/vnc.py:256
msgid "The VNC server is now running."
msgstr "Le servitor VNC es nunc active."

#: pyanaconda/vnc.py:260
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Tu ha eligite pro connecter a un vncviewer ascoltante. \n"
"Isto non require le definition de un contrasigno. Si tu \n"
"defini un contrasigno, illo essera usate si le connexion \n"
"al vncviewer non succede.\n"
"\n"

#: pyanaconda/vnc.py:265
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=PASSWORD boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:269
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Tu eligite pro executar VNC con un contrasigno. \n"
"\n"

#: pyanaconda/vnc.py:271
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Error incognite. Abandono. \n"
"\n"

#: pyanaconda/vnc.py:293
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""
"Le longitude de contrasigno VNC debe esser de 6 a 8 characteres.\n"
"Introduce un nove contrasigno o lassa lo in blanco pro nulle contrasigno."

#: pyanaconda/display.py:291
msgid "Graphical installation is not available. Starting text mode."
msgstr "Le installation graphic non es disponibile. Le modo textual se initia."

#: pyanaconda/display.py:299
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""
"Le modo textual provide optiones de installation limitate. Illo non permitte "
"specificar un proprie arrangiamento de partitiones sur le disco. Velle tu "
"usar le modo VNC in su loco?"

#: pyanaconda/display.py:328
msgid ""
"X was unable to start on your machine. Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""

#: pyanaconda/installation.py:82
msgid "Configuring installed system"
msgstr "Configuration del systema installate"

#: pyanaconda/installation.py:114
msgid "Writing network configuration"
msgstr "Creation del configuration de rete"

#: pyanaconda/installation.py:120
msgid "Creating users"
msgstr "Creation de usatores"

#: pyanaconda/installation.py:127
msgid "Configuring addons"
msgstr "Configuration del additiones"

#: pyanaconda/installation.py:143
msgid "Generating initramfs"
msgstr "Generation de initramfs"

#: pyanaconda/installation.py:164
msgid "Running post-installation scripts"
msgstr "Execution de scripts post installation"

#: pyanaconda/installation.py:170
msgid "Setting up kexec"
msgstr ""

#: pyanaconda/installation.py:175
msgid "Storing configuration files and kickstarts"
msgstr ""

#: pyanaconda/installation.py:219
#, python-format
msgid "Waiting for %s threads to finish"
msgstr "Periodo de attender pro le fin de %s filos"

#: pyanaconda/installation.py:232
msgid "Setting up the installation environment"
msgstr "Configuration del ambiente de installation"

#: pyanaconda/installation.py:249 pyanaconda/installation.py:317
msgid "Configuring storage"
msgstr ""

#: pyanaconda/installation.py:259 pyanaconda/kickstart.py:600
msgid "Running pre-installation scripts"
msgstr "Execution del scripts de pre-installation"

#: pyanaconda/installation.py:266
msgid "Running pre-installation tasks"
msgstr "Execution del cargas de pre-installation"

#: pyanaconda/installation.py:311
msgid "Installing."
msgstr ""

#: pyanaconda/installation.py:324
msgid "Installing boot loader"
msgstr ""

#: pyanaconda/installation.py:345
msgid "Creating snapshots"
msgstr ""

#: pyanaconda/platform.py:52
msgid "boot loader device"
msgstr ""

#: pyanaconda/platform.py:54
msgid "RAID Device"
msgstr ""

#: pyanaconda/platform.py:55
msgid "First sector of boot partition"
msgstr ""

#: pyanaconda/platform.py:122 pyanaconda/platform.py:241
#: pyanaconda/platform.py:257
msgid "Master Boot Record"
msgstr ""

#: pyanaconda/platform.py:129
msgid ""
"You must include at least one MBR- or GPT-formatted disk as an install "
"target."
msgstr ""

#: pyanaconda/platform.py:146
msgid "EFI System Partition"
msgstr "Partition de systema EFI"

#: pyanaconda/platform.py:152
msgid ""
"For a UEFI installation, you must include an EFI System Partition on a GPT-"
"formatted disk, mounted at /boot/efi."
msgstr ""

#: pyanaconda/platform.py:166
msgid "Apple EFI Boot Partition"
msgstr ""

#: pyanaconda/platform.py:169
msgid ""
"For a UEFI installation, you must include a Linux HFS+ ESP on a GPT-"
"formatted disk, mounted at /boot/efi."
msgstr ""

#: pyanaconda/platform.py:201
msgid "PReP Boot Partition"
msgstr ""

#: pyanaconda/platform.py:203
msgid ""
"You must include a PReP Boot Partition within the first 4GiB of an MBR- or "
"GPT-formatted disk."
msgstr ""

#: pyanaconda/platform.py:215
msgid "Apple Bootstrap Partition"
msgstr ""

#: pyanaconda/platform.py:218
msgid ""
"You must include an Apple Bootstrap Partition on an Apple Partition Map-"
"formatted disk."
msgstr ""

#: pyanaconda/platform.py:230
msgid "You must include at least one disk as an install target."
msgstr ""

#: pyanaconda/platform.py:240
msgid "DASD"
msgstr ""

#: pyanaconda/platform.py:242
msgid "zFCP"
msgstr ""

#: pyanaconda/platform.py:247
msgid ""
"You must include at least one MBR- or DASD-formatted disk as an install "
"target."
msgstr ""

#: pyanaconda/platform.py:261
msgid "You must include at least one MBR-formatted disk as an install target."
msgstr ""

#: pyanaconda/product.py:56
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr "INSTALLATION DE %(productName)s %(productVersion)s"

#: pyanaconda/pwpolicy.py:146
#, python-format
msgid "policy name required for %s"
msgstr ""

#: pyanaconda/pwpolicy.py:155
#, python-format
msgid "A %(command)s with the name %(policyName)s has already been defined."
msgstr ""

#: pyanaconda/rescue.py:272
msgid "Unable to find /bin/bash to execute!  Not starting shell."
msgstr ""

#: pyanaconda/rescue.py:289
msgid "Rescue"
msgstr "Salvar"

#: pyanaconda/rescue.py:296
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory : %s.  You can then make any changes required "
"to your system.  Choose '1' to proceed with this step.\n"
"You can choose to mount your file systems read-only instead of read-write by "
"choosing '2'.\n"
"If for some reason this process does not work choose '3' to skip directly to "
"a shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:308
msgid "Continue"
msgstr "Continuar"

#: pyanaconda/rescue.py:309
msgid "Read-only mount"
msgstr ""

#: pyanaconda/rescue.py:310
msgid "Skip to shell"
msgstr ""

#: pyanaconda/rescue.py:311
msgid "Quit (Reboot)"
msgstr ""

#: pyanaconda/rescue.py:408
msgid "Rescue Shell"
msgstr ""

#: pyanaconda/rescue.py:418
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr "Exeque %s pro dismontar le systema al fin."

#: pyanaconda/rescue.py:419
msgid ""
"When finished, please exit from the shell and your system will reboot.\n"
msgstr ""

#: pyanaconda/rescue.py:430
#, python-format
msgid ""
"Your system has been mounted under %(mountpoint)s.\n"
"\n"
"If you would like to make the root of your system the root of the active "
"system, run the command:\n"
"\n"
"\tchroot %(mountpoint)s\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:442
#, python-brace-format
msgid ""
"An error occurred trying to mount some or all of your system: {message}\n"
"\n"
"Some of it may be mounted under {path}."
msgstr ""

#: pyanaconda/rescue.py:451
msgid "Rebooting."
msgstr ""

#: pyanaconda/rescue.py:454
#, python-format
msgid "You don't have any Linux partitions. %s\n"
msgstr ""

#: pyanaconda/rescue.py:460
msgid "Not mounting the system.\n"
msgstr ""

#: pyanaconda/rescue.py:474
#, python-format
msgid "Please press %s to get a shell"
msgstr "Pressa sur %s pro obtener un interpretator"

#: pyanaconda/rescue.py:491
msgid "Root Selection"
msgstr ""

#: pyanaconda/rescue.py:517
msgid "The following installations were discovered on your system."
msgstr ""

#: pyanaconda/errors.py:112
msgid "An error occurred while activating your storage configuration."
msgstr ""
"Un error occurreva durante le activation del configuration del immagazinage."

#: pyanaconda/errors.py:119
msgid ""
"There is a problem with your existing storage configuration.You must resolve "
"this matter before the installation can proceed. There is a shell available "
"for use which you can access by pressing ctrl-alt-f1 and then ctrl-b 2.\n"
"\n"
"Once you have resolved the issue you can retry the storage scan. If you do "
"not fix it you will have to exit the installer."
msgstr ""

#: pyanaconda/errors.py:128
msgctxt "GUI|Storage Detailed Error Dialog"
msgid "_Exit Installer"
msgstr "_Exir del installator"

#: pyanaconda/errors.py:129
msgctxt "GUI|Storage Detailed Error Dialog"
msgid "_Retry"
msgstr "_Reprobar"

#: pyanaconda/errors.py:136
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""
"Le imagine ISO %s ha un dimension que non es un multiple de 2048 octettos. "
"Ce pote significar que illo era corrumpite durante le transferentia a iste "
"computator.\n"
"\n"
"Il esrecommendante "

#: pyanaconda/errors.py:150
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:161
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:167
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:172
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:193
msgid ""
"Some packages, groups or modules are broken, the installation will be "
"aborted."
msgstr ""

#: pyanaconda/errors.py:199
msgid "Would you like to ignore this and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:206
#, python-format
msgid ""
"Stream was not specified for a module without a default stream. This is a "
"fatal error and installation will be aborted. The details of this error "
"are:\n"
"\n"
"%(exception)s"
msgstr ""

#: pyanaconda/errors.py:214
#, python-format
msgid ""
"Multiple streams have been specified for a single module. This is a fatal "
"error and installation will be aborted. The details of this error are:\n"
"\n"
"%(exception)s"
msgstr ""

#: pyanaconda/errors.py:222
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:230
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:238
msgid ""
"The following software marked for installation has errors.\n"
"This is likely caused by an error with your installation source."
msgstr ""

#: pyanaconda/errors.py:246
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:258
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/core/kickstart/addon.py:128
msgid "Missing name of the %addon section."
msgstr ""

#: pyanaconda/core/kickstart/addon.py:138
msgid "Unknown name of the %addon section."
msgstr ""

#: pyanaconda/core/constants.py:92
msgid "Start VNC"
msgstr "Lancear VNC"

#: pyanaconda/core/constants.py:93
msgid "Use text mode"
msgstr "Usar modo textual"

#: pyanaconda/core/constants.py:96
msgid "Do you really want to quit?"
msgstr "Quitar realmente?"

#: pyanaconda/core/constants.py:162
msgid ""
"Support for this hardware has been removed in this major OS release. Please "
"check theremoved functionality section of the release notes."
msgstr ""

#: pyanaconda/core/constants.py:167
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat. For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/core/constants.py:173
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""
"Nulle discos discoperite. Extingue le computator, connecte al minus un disco "
"e reinitia pro ultimar le installation."

#: pyanaconda/core/constants.py:178
msgid "No disks selected; please select at least one disk to install to."
msgstr "Nulle discos seligite; selige al minus un disco pro le installation."

#: pyanaconda/core/constants.py:190
msgid "The password is empty."
msgstr "Le contrasigno es vacue."

#: pyanaconda/core/constants.py:191
msgid "The passphrase is empty."
msgstr ""

#: pyanaconda/core/constants.py:194
msgid "The passwords do not match."
msgstr "Le contrasignos non es correspondente."

#: pyanaconda/core/constants.py:195
msgid "The passphrases do not match."
msgstr "Le phrase de contrasignos non coincide."

#: pyanaconda/core/constants.py:198
msgid "The passwords you entered were different. Please try again."
msgstr ""

#: pyanaconda/core/constants.py:199
msgid "The passphrases you entered were different. Please try again."
msgstr ""

#: pyanaconda/core/constants.py:205
msgid "The password is too short"
msgstr ""

#: pyanaconda/core/constants.py:206
msgid "The passphrase is too short"
msgstr ""

#: pyanaconda/core/constants.py:209
msgid "The password you have provided is weak."
msgstr ""

#: pyanaconda/core/constants.py:210
msgid "The passphrase you have provided is weak."
msgstr ""

#: pyanaconda/core/constants.py:213
msgid "The password you have provided is weak:"
msgstr ""

#: pyanaconda/core/constants.py:214
msgid "The passphrase you have provided is weak:"
msgstr ""

#: pyanaconda/core/constants.py:216
msgid "Press <b>Done</b> again to use the password anyway."
msgstr "Pressa sur <b>Facite</b> de nove pro usar le contrasigno comocunque."

#: pyanaconda/core/constants.py:218
msgid ""
"The password you have provided contains non-ASCII characters. You may not be "
"able to switch between keyboard layouts when typing it."
msgstr ""

#: pyanaconda/core/constants.py:219
msgid ""
"The passphrase you have provided contains non-ASCII characters. You may not "
"be able to switch between keyboard layouts when typing it."
msgstr ""

#: pyanaconda/core/constants.py:221
msgid "You will have to press <b>Done</b> twice to confirm it."
msgstr "Tu debera pressar sur <b>Facite</b> duo vices pro confirmar lo."

#: pyanaconda/core/constants.py:222
msgid "Password set."
msgstr "Le contrasigno es definite."

#: pyanaconda/core/constants.py:226
msgid "Empty"
msgstr "Vacue"

#: pyanaconda/core/constants.py:227
msgid "Too short"
msgstr ""

#: pyanaconda/core/constants.py:228
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:182
msgid "Weak"
msgstr "Insecur"

#: pyanaconda/core/constants.py:229
msgid "Fair"
msgstr "Passabile"

#: pyanaconda/core/constants.py:230
msgid "Good"
msgstr "Bon"

#: pyanaconda/core/constants.py:231
msgid "Strong"
msgstr "Secur"

#: pyanaconda/core/constants.py:234
msgid "Hide password."
msgstr "Celar contrasigno."

#: pyanaconda/core/constants.py:235
msgid "Show password."
msgstr "Monstrar contrasigno."

#: pyanaconda/core/constants.py:292
msgid "Probing storage..."
msgstr "Probation de immagazinage..."

#: pyanaconda/core/constants.py:293
msgid "Testing availability..."
msgstr ""

#: pyanaconda/core/constants.py:294
msgid "Downloading package metadata..."
msgstr "Discargamento de metadatos de pacchettos..."

#: pyanaconda/core/constants.py:295
msgid "Downloading group metadata..."
msgstr "Discargamento de metadatos de gruppos..."

#: pyanaconda/core/constants.py:298
msgid "Anaconda Installer"
msgstr "Installator Anaconda"

#: pyanaconda/core/util.py:747
msgid "No host url"
msgstr "Nulle url de hospite"

#: pyanaconda/core/util.py:768
msgid "malformed URL, cannot parse it."
msgstr "URL mal formate, non pote analysar lo."

#: pyanaconda/core/util.py:784
msgid "URL has no host component"
msgstr "URL non ha componente de hospite"

#: pyanaconda/core/users.py:66
#, python-format
msgid "User name is reserved for system: %s"
msgstr ""

#: pyanaconda/core/users.py:90
msgid ""
"Either a group name in the group list is invalid or groups are not separated "
"by a comma."
msgstr ""

#: pyanaconda/core/users.py:122
msgid "Name cannot start with '-' character."
msgstr ""

#: pyanaconda/core/users.py:125
#, python-format
msgid "Name '%s' is not allowed."
msgstr ""

#: pyanaconda/core/users.py:128
msgid "Fully numeric name is not allowed."
msgstr ""

#: pyanaconda/core/users.py:132
msgid "Name '$' is not allowed."
msgstr ""

#: pyanaconda/core/users.py:142
#, python-format
msgid "Name cannot contain character: '%s'"
msgstr ""

#: pyanaconda/core/users.py:145
msgid "Name must be shorter than 33 characters."
msgstr ""

#: pyanaconda/core/users.py:149
#, python-format
msgid "Name '%s' is invalid."
msgstr ""

#: pyanaconda/addons.py:97
#, python-format
msgid "Executing %s addon"
msgstr ""

#: pyanaconda/addons.py:113
#, python-format
msgid "Setting up %s addon"
msgstr ""

#: pyanaconda/modules/network/firewall/installation.py:102
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr "%s es mancante. Impossibile de configurar le parafoco."

#: pyanaconda/modules/storage/kickstart.py:65
msgid "File system type \"{}\" given in autopart command is invalid."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:87
msgid ""
"Disklabel \"{}\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:93
msgid "Disk \"{}\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:98
msgid "Device \"{}\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:117
#: pyanaconda/modules/storage/kickstart.py:122
msgid "Disk \"{}\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:132
msgid "NIC \"{}\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:157
#, python-brace-format
msgid ""
"Network interface \"{nic}\" required by iSCSI \"{iscsi_target}\" target is "
"not up."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:173
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:211
#: pyanaconda/modules/storage/kickstart.py:223
msgid "Namespace \"{}\" given in nvdimm command was not found."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:228
msgid "Disk \"{}\" given in nvdimm command does not exist."
msgstr ""

#: pyanaconda/modules/storage/kickstart.py:241
msgid ""
"Incorrectly specified origin of the snapshot. Use format \"VolGroup/LV-name\""
msgstr ""

#: pyanaconda/modules/storage/bootloader/bootloader.py:160
msgid "GRUB2 does not support installation to a partition."
msgstr "GRUB2 non supporta installation in un partition."

#: pyanaconda/modules/storage/bootloader/bootloader.py:167
msgid "GRUB2 encrypted password must be in grub.pbkdf2 format."
msgstr ""

#: pyanaconda/modules/storage/installation.py:80
msgid "An error occurred while resizing the device {}: {}"
msgstr "Un error occurreva durante le redimensionamento del dispositivo {}: {}"

#: pyanaconda/modules/storage/installation.py:131
msgid "Gathering entropy 100%"
msgstr ""

#: pyanaconda/modules/storage/installation.py:135
msgid "Gathering entropy (time ran out)"
msgstr ""

#: pyanaconda/modules/storage/installation.py:138
#, python-brace-format
msgid "Gathering entropy {percents}% (remaining time {time})"
msgstr ""

#: pyanaconda/modules/storage/partitioning/manual/manual_partitioning.py:64
#, python-format
msgid "Unknown or invalid device '%s' specified"
msgstr ""

#: pyanaconda/modules/storage/partitioning/manual/manual_partitioning.py:72
#, python-format
msgid ""
"Unknown or invalid format '%(format)s' specified for device '%(device)s'"
msgstr ""

#: pyanaconda/modules/storage/partitioning/manual/manual_partitioning.py:79
#, python-format
msgid "No format on device '%s'"
msgstr ""

#: pyanaconda/modules/storage/partitioning/automatic/utils.py:128
msgid "No usable disks selected."
msgstr "Nulle discos usabile seligite."

#: pyanaconda/modules/storage/partitioning/automatic/utils.py:131
msgid "Not enough free space on selected disks."
msgstr "Il non ha satis de spatio libere sur le discos seligite."

#: pyanaconda/modules/storage/partitioning/automatic/utils.py:273
msgid "No big enough free space on disks for automatic partitioning"
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:160
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:201
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:214
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:520
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:227
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:532
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:252
#, python-format
msgid "The size \"%s\" is invalid."
msgstr "Le grandor \"%s\" es invalide."

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:261
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:268
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:378
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:281
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:292
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:389
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:970
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:981
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1046
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:313
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:615
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1021
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:329
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:348
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:334
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:342
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:359
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1091
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr "Le grandor maxime \"%s\" es invalide."

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:549
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:915
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1257
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:558
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:565
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:590
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:597
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:638
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:649
#, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:767
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:773
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:781
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""
"gruppo de tomo \"%s\" definite sin alcun tomos physic. O specifica tomos "
"physic o usa --useexisting."

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:794
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:807
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""
"volgroup --noformat e volgroup --useexisting debe etiam usar le option --"
"name= ."

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:814
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr "Le gruppo de tomo \"%s\" date in le commando volgroup non existe."

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:819
#, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr "Le nomine de gruppo de volumine “%s” es ja in uso."

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:923
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:932
msgid "Cache PVs must belong to the same VG as the cached LV"
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:941
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:950
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:957
#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1034
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:997
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1004
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1223
#, python-format
msgid ""
"Btrfs partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"btrfs\"."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1231
#, python-format
msgid "Tried to use undefined partition \"%s\" in Btrfs volume specification."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1246
msgid ""
"Btrfs volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py:1274
#, python-format
msgid "Btrfs volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:205
#, python-brace-format
msgid "New {name} {version} Installation"
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:281
msgid "Cannot relabel already existing file system."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:284
msgid "Cannot set label on file system."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:287
msgid "Unacceptable label format for file system."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:302
msgid "That mount point is already in use. Try something else?"
msgstr "Iste puncto de montage ja es in uso. Probar qualcosa altere?"

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:305
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:308
#: pyanaconda/modules/storage/partitioning/interactive/utils.py:320
msgid "That mount point is invalid. Try something else?"
msgstr "Iste puncto de montage non es valide. Probar qualcosa altere?"

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:335
msgid "Invalid container name."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:363
#, python-brace-format
msgid ""
"The RAID level you have selected ({level}) requires more disks ({min}) than "
"you currently have selected ({count})."
msgstr ""
"Le nivello RAID seligite ({level}) require plus de discos ({min}) que le "
"actualmente seligite ({count})."

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:405
msgid "Please enter a mount point."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:418
#, python-brace-format
msgid "/boot/efi must be on a device of type {type} or {another}"
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:425
#, python-brace-format
msgid "{fs} must be on a device of type {type}"
msgstr "{fs} debe esser sur un dispositivo del typo {type}"

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:431
#: pyanaconda/modules/storage/partitioning/interactive/utils.py:434
msgid "{} cannot be encrypted"
msgstr "{} non pote esser cryptate"

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:441
msgid "Device does not support RAID level selection {}."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:1122
#, python-format
msgid ""
"This Software RAID array is missing %(missing)d of %(total)d member "
"partitions. You can remove it or select a different device."
msgstr ""

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:1129
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""
"%(missingPVs)d de %(totalPVs)d volumines physic manca in iste gruppo de "
"volumines LVM. On pote remover lo o seliger un altere dispositivo."

#: pyanaconda/modules/storage/partitioning/interactive/utils.py:1133
#, python-format
msgid ""
"This %(type)s device is missing member devices. You can remove it or select "
"a different device."
msgstr ""

#: pyanaconda/modules/storage/snapshot/device.py:43
#, python-format
msgid "Snapshot: origin \"%s\" doesn't exists!"
msgstr ""

#: pyanaconda/modules/storage/snapshot/device.py:47
#, python-format
msgid ""
"Snapshot: origin \"%(origin)s\" of snapshot \"%(name)s\" is not a valid thin "
"LV device."
msgstr ""

#: pyanaconda/modules/storage/snapshot/device.py:53
#, python-format
msgid "Snapshot %s already exists."
msgstr ""

#: pyanaconda/modules/storage/devicetree/viewer.py:302
#, python-brace-format
msgid "{device_description} ({device_name})"
msgstr ""

#: pyanaconda/modules/storage/devicetree/viewer.py:308
#, python-brace-format
msgid "{device_name} on {container_name}"
msgstr ""

#: pyanaconda/modules/payloads/kickstart.py:42
msgid "The %packages --ignorebroken feature is not supported on your product!"
msgstr ""

#: pyanaconda/kickstart.py:551
msgid ""
"\n"
"Some warnings occurred during reading the kickstart file:"
msgstr ""

#: pyanaconda/kickstart.py:556
#, python-format
msgid ""
"\n"
"An error occurred during reading the kickstart file:\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""

#: anaconda.py:381
msgid "Unable to create PID file"
msgstr ""

#: anaconda.py:382
#, python-format
msgid ""
"Anaconda is unable to create %s because the file already exists. Anaconda is "
"already running, or a previous instance of anaconda has crashed."
msgstr ""

#: widgets/src/BaseWindow.c:147
msgid "DISTRIBUTION INSTALLATION"
msgstr "INSTALLATION DEL DISTRIBUTION"

#: widgets/src/BaseWindow.c:148
msgid "SPOKE NAME"
msgstr "NOMINE DEL SUCCURSAL"

#: widgets/src/BaseWindow.c:149
msgid "PRE-RELEASE / TESTING"
msgstr "PRE-EDITION / PROBATION"

#: widgets/src/DiskOverview.c:97
msgid "New Device"
msgstr "Nove dispositivo"

#: widgets/src/DiskOverview.c:99 widgets/src/DiskOverview.c:100
msgid "0 MB"
msgstr "0 MB"

#: widgets/src/LayoutIndicator.c:33
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""
"Le disposition actual: '%s'. Clicca pro cambiar lo al disposition sequente."

#: widgets/src/LayoutIndicator.c:34
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""
"Le disposition actual: '%s'. Adde dispositiones pro permitter cambiamentos."

#: widgets/src/MountpointSelector.c:84
msgid "0 GB"
msgstr "0 GB"

#: widgets/src/SpokeWindow.c:55
msgctxt "GUI|Spoke Navigation"
msgid "_Done"
msgstr "_Facite"

#: widgets/src/StandaloneWindow.c:52
msgctxt "GUI|Standalone Navigation"
msgid "_Quit"
msgstr "_Quitar"

#: widgets/src/StandaloneWindow.c:53
msgctxt "GUI|Standalone Navigation"
msgid "_Continue"
msgstr "_Continuar"

#: widgets/src/SpokeSelector.c:82
msgid "New Selector"
msgstr "Nove selector"

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:145
msgid "INSTALLATION SUMMARY"
msgstr "SUMMARIO DE INSTALLATION"

#: pyanaconda/ui/gui/hubs/summary.glade:81
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/hubs/summary.glade:97
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr "_Comenciar le installation"

#: pyanaconda/ui/gui/hubs/summary.glade:128
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr "Selige le interfacie de rete que es connectite al commutator FCoE."

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr "_NIC:"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr "Usar _DCB"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr "_Adder disco(s) FCoE"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr "Error al addition de SAN FCoE."

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""
"Le sequente error occurreva durante le tentativa de adder un SAN FCoE. "
"Verifica le configuration e reproba"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:82
msgid "ADD zFCP STORAGE TARGET"
msgstr "ADDER SCOPO DE IMMAGAZINAGE zFCP"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:105
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:132
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr "Numero de _dispositivo:"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:159
msgid "WWPN:"
msgstr "WWPN:"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:177
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr "_Initiar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:214
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr "Discoperimento de dispositivos zFCP. Isto pote prender un momento…"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:253
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:212
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:268
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""
"Le sequente error occurreva durante le discoperimento del dispositivos zFCP. "
"Verifica le information de configuration e reproba."

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:291
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr "_Reprobar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:346
#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:307
msgid "Device discovery succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:373
msgid "LUN:"
msgstr "LUN:"

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:23
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:70
msgid ""
"The following NVDIMM devices selected from the list will be reconfigured to "
"sector mode:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:85
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "_Sector size:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:103
msgctxt "GUI|Advanced Storage|NVDIMM|Reconfigure"
msgid "Start _Reconfiguration"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:139
msgid "Reconfiguring NVDIMM devices. This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:205
msgid "The following error occurred reconfiguring NVDIMM devices:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:252
msgid "Devices reconfiguration succeeded."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:278
msgid ""
"Rescanning disks after reconfiguration.\n"
"This may take a moment ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/nvdimm.glade:334
msgid "Warning: Existing data on reconfigured devices will be lost."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:40
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:54
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:85
msgid "ADD iSCSI STORAGE TARGET"
msgstr "ADDER SCOPO DE IMMAGAZINAGE iSCSI"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:114
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""
"Pro usar discos iSCSI, on debe fornir le adresse de scopo iSCSI e le nomine "
"de initiator iSCSI configurate pro le host."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:142
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr "Adresse IP del _scopo:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:170
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr "Nomine de _initiator iSCSI:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:760
msgid "No credentials (discovery authentication disabled)"
msgstr "Nulle datos de authentication (discoperta es disactivate)"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:188
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:761
msgid "CHAP pair"
msgstr "Par de CHAP"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:189
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:762
msgid "CHAP pair and a reverse pair"
msgstr "Par de CHAP e un par reverse"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:203
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr "Typo de authentication de _discoperimento:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:218
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr ""
"<span size=\"small\"><b>Exemplo:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:263
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:277
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:389
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:403
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:417
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:431
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "_Contrasigno CHAP reverse:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:465
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr "_Initiar discoperimento"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:502
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr "Discoperimento de scopos iSCSI. Isto pote prender un momento…"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:541
msgid "Discovery login rejected."
msgstr "Insuccesso de authentication al discoperimento."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:556
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""
"Le sequente error occurreva durante le discoperimento del scopos iSCSI. "
"Verifica le information de authentication e reproba."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:579
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr "_Tentar le discoperimento de novo"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:619
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:695
msgid "Node Name"
msgstr "Nomine de nodo"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:706
msgid "Interface"
msgstr "Interfacie"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:717
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:745
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:763
msgid "Use the credentials from discovery"
msgstr "Usar le datos de authentication desde discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:797
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:811
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:923
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:937
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Contra_signo CHAP:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:951
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:965
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "_Contrasigno CHAP reverse:"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1014
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr "_Authenticar se"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1069
msgid "Node login failed."
msgstr "Impossibile de authenticar se al nodo."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1084
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""
"Le sequente error ha occurrite durante de authentication al nodo iSCSI. "
"Verifica le information de authentication e tenta de novo."

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1107
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr "_Authenticar se de novo"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:22
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:36
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:67
msgid "ADD DASD STORAGE TARGET"
msgstr "ADDER SCOPO DE IMMAGAZINAGE DASD"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:90
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:117
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr "Numero de _dispositivo:"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:136
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr "_Initiar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:173
msgid "Discovering DASD devices.  This may take a moment..."
msgstr "Discoperimento de dispositivos DASD. Isto pote prender un momento…"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:227
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""
"Le sequente error occurreva durante le discoperimento del dispositivos DASD. "
"Verifica le information de configuration e reproba."

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:250
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr "_Reprobar discoperta"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:324
msgid "The added DASD can be found under \"Local Standard Disks.\""
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr "_Salveguardar le phrase de contrasigno"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:65
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr "PHRASE DE CONTRASIGNO PRO CRYPTAR LE DISCO"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:81
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""
"Tu ha eligite a cryptar certe datos. Tu debe crear un phrase de contrasigno "
"que essera usate pro acceder al datos post le initio del computator."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:102
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr "_Phrase de contrasigno:"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:133
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:275
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""
"Aviso: Le cambiamento del disposition de claviero (del disposition "
"predefinite) non essera possibile si le discos necessita decryptation post "
"le installation."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
msgid "RESCAN DISKS"
msgstr "RESCANNAR DISCOS"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""
"Ora on pote remover o inserer dispositivos additional e pressar sur "
"'Rescannar discos' in basso pro effectuar le cambiamentos."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""
"<b>Aviso:</b> Tote le cambiamentos de immagazinage facite per le uso de "
"installator essera perdite quando on pressa sur 'Rescannar discos'."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr "_Rescannar discos"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr "Scannation de discos.  Isto pote prender un momento..."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""
"Il pote <a href=\"\">retornar al menu principal</a> pro completar altere\n"
"optiones de installation durante le scannation."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr "Le operation a rescannar discos se ha completate."

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""
"Pressar sur 'OK' in basso va transferer al schermo a seliger discos ubi "
"illos debe esser seligite de novo."

#: pyanaconda/ui/gui/spokes/lib/summary.glade:25
#: pyanaconda/ui/gui/spokes/lib/summary.glade:89
msgid "SUMMARY OF CHANGES"
msgstr "SUMMARIO DE CAMBIAMENTOS"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:42
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr "_Cancellar e retornar al partitionamento personalisate"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:57
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr "_Acceptar cambiamentos"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:105
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""
"Le personalisationes effectuara le sequente cambiamentos post le retorno al "
"pagina principal e le comenciamento de installation:"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:133
msgid "Order"
msgstr "Ordine"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:144
#: pyanaconda/ui/gui/spokes/lib/resize.glade:194
msgid "Action"
msgstr "Action"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:155
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1775
msgid "Type"
msgstr "Typo"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:166
msgid "Device"
msgstr "Dispositivo"

#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:23
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr "_Quitar"

#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:37
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr "_Continuar"

#: pyanaconda/ui/gui/spokes/lib/unsupported_hardware.glade:122
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:53
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:68
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr "_Recuperar spatio"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:101
msgid "RECLAIM DISK SPACE"
msgstr "RECUPERAR SPATIO DE DISCO"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:118
msgid "Description goes here."
msgstr "Le description es hic."

#: pyanaconda/ui/gui/spokes/lib/resize.glade:161
#: pyanaconda/ui/gui/spokes/lib/cart.glade:131
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:238
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:735
#: pyanaconda/ui/gui/spokes/installation_source.glade:1021
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:386
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1438
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1761
msgid "Name"
msgstr "Nomine"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:172
msgid "File System"
msgstr "Systema de files"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:183
msgid "Reclaimable Space"
msgstr "Spatio recuperabile"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:219
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr "_Preservar"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:235
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr "_Deler"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:251
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr "_Contraher"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:365
msgid "Reclaim"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:24
#: pyanaconda/ui/gui/spokes/lib/cart.glade:76
msgid "SELECTED DISKS"
msgstr "DISCOS SELIGITE"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:41
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr "_Clauder"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:104
msgid "Boot"
msgstr "Boot"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:118
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:220
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:722
msgid "Description"
msgstr "Description"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:143
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:251
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:748
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:414
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:814
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1148
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1466
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1789
msgid "Capacity"
msgstr "Capacitate"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:156
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:264
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:761
msgid "Free"
msgstr "Libere"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:197
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr "_Remover"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:224
msgid "Disk summary goes here"
msgstr "Le summario de disco es hic"

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr "_Deler lo"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:127
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:142
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr "_Seliger"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:173
msgid "CONFIGURE MOUNT POINT"
msgstr "CONFIGURAR PUNCTOS DE MONTAGE"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:189
msgid "Select one or more disks this device may reside on."
msgstr "Selige un o alicun discos ubi iste dispositivo pote resider."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:324
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:339
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr "Adder le _puncto de montage"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:373
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Desired _Capacity:"
msgstr "Capacitate _desirate:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:390
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr "_Puncto de montage:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:408
msgid "ADD A NEW MOUNT POINT"
msgstr "ADDER UN NOVE PUNCTO DE MONTAGE"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:426
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""
"Plus del optiones de personalisation es disponibile\n"
"post creation del puncto de montage in basso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:461
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""
"Iste puncto de montage es ja in \n"
"uso. Esque tu tenta un altere cosa?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:485
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr "e.g.: \"20 GB\", \"500mb\" (minus le virgulettas)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:542
msgid "Single"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:546
#: pyanaconda/ui/gui/spokes/custom_storage.glade:56
msgid "RAID0"
msgstr "RAID0"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:550
#: pyanaconda/ui/gui/spokes/custom_storage.glade:60
msgid "RAID1"
msgstr "RAID1"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:554
#: pyanaconda/ui/gui/spokes/custom_storage.glade:64
msgid "RAID4"
msgstr "RAID4"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:558
#: pyanaconda/ui/gui/spokes/custom_storage.glade:68
msgid "RAID5"
msgstr "RAID5"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:562
#: pyanaconda/ui/gui/spokes/custom_storage.glade:72
msgid "RAID6"
msgstr "RAID6"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:566
#: pyanaconda/ui/gui/spokes/custom_storage.glade:76
msgid "RAID10"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:590
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:604
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr "_Salveguardar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:636
msgid "CONFIGURE CONTAINER"
msgstr "CONFIGURAR LE CONTAINER"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:652
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr "Crea un nomine pro iste container e selige al minus un disco in basso."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:670
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr "No_mine:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:789
msgid "RAID Level:"
msgstr "Nivello de RAID:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:818
msgid "Encrypt"
msgstr "Cryptar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:859
msgid "Please enter a valid name."
msgstr "Introduce un nomine valide."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:889
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr "Regulamento de _dimension:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:904
msgid "Automatic"
msgstr "Automatic"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:905
msgid "As large as possible"
msgstr "Le plus grande possibile"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:906
msgid "Fixed"
msgstr "Fixate"

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""
"Un error incognite ha occurrite durante le installation. Plus de detalios es "
"in basso."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr "DASDS NON FORMATATE"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted or LDL DASDs have been detected on your system. "
"You can choose to format them now with dasdfmt or cancel to leave them "
"unformatted. Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
msgid "Unformatted DASDs Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""
"<b>Aviso</b>: Tote le cambiamentos de immagazinage facite per le uso de "
"installator essera perdite quando on elige formatar."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:209
msgid ""
"This may take a moment. You may <a href=\"\">go back to the main menu</a> to "
"complete other installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:248
msgid "Disk formatting complete."
msgstr "Formatation de disco complete."

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:108
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:122
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:147
msgid "Add and mark for usage NTP servers"
msgstr "Adder servitores NTP e marcar los pro le uso"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:169
msgid "New NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:188
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:208
msgid "This URL refers to a pool of NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:254
msgid "Pool"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:267
msgid "Working"
msgstr "Operational"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:275
msgid "Use"
msgstr "Usar"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:303
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:517
msgid "Configure NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:324
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:932
msgid "TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:374
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr "_Region:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:404
msgid "Region"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:421
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr "_Citate:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:451
msgid "City"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:479
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr "Hora de re_te"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:497
msgid "Use Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:584
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:597
msgctxt "GUI|Date and Time"
msgid ":"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:617
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:635
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:653
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:671
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:689
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:733
msgid "AM/PM Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:769
msgid "AM/PM Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:820
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr "24 _horas"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:838
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr "_AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:897
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:388
msgid "KEYBOARD LAYOUT"
msgstr "DISPOSITION DE CLAVIERO"

#: pyanaconda/ui/gui/spokes/keyboard.glade:56
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""
"Qual dispositiones de claviero debe usar se in iste systema? Move le "
"disposition in alto del lista pro usar lo per predefinition."

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""
"Cambiamentos hic affectara solmente le systema installate. Usa le "
"instrumentos de scriptorio pro configurar le claviero pro le processo de "
"installation."

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
msgid "Selected Layouts"
msgstr "Dispositiones seligite"

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr "_Adder disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr "_Remover disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr "Mover le disposition seligite in al_to"

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr "Mover le disposition seligite in _basso"

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr "_Previder disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr "_Probar le disposition seligite in basso:"

#: pyanaconda/ui/gui/spokes/keyboard.glade:331
msgid "Alt + Shift to switch layouts."
msgstr "Alt + Shift pro cambiar dispositiones."

#: pyanaconda/ui/gui/spokes/keyboard.glade:346
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr "_Optiones"

#: pyanaconda/ui/gui/spokes/keyboard.glade:421
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/keyboard.glade:435
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr "_Adder"

#: pyanaconda/ui/gui/spokes/keyboard.glade:467
msgid "ADD A KEYBOARD LAYOUT"
msgstr "ADDER UN DISPOSITION DE CLAVIERO"

#: pyanaconda/ui/gui/spokes/keyboard.glade:483
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""
"Le disposition de claviero pote esser addite per le selection in basso:"

#: pyanaconda/ui/gui/spokes/keyboard.glade:532
msgid "Available Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:573
msgid "Add Layout"
msgstr "Adder disposition"

#: pyanaconda/ui/gui/spokes/keyboard.glade:605
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/keyboard.glade:619
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/keyboard.glade:651
msgid "LAYOUT SWITCHING OPTIONS"
msgstr "OPTIONES DEL CAMBIAMENTO DE DISPOSITION"

#: pyanaconda/ui/gui/spokes/keyboard.glade:668
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""
"Qual combination(es) de claves debe usar se pro cambiar dispositiones de "
"clavieros?"

#: pyanaconda/ui/gui/spokes/keyboard.glade:751
msgid "Layout Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/language_support.glade:33
msgid "LANGUAGE SUPPORT"
msgstr "SUPPORTO DE LINGUAS"

#: pyanaconda/ui/gui/spokes/language_support.glade:78
msgid "Select additional language support to be installed:"
msgstr "Selige le supporto pro linguas additional que debe esser installate:"

#: pyanaconda/ui/gui/spokes/language_support.glade:186
#: pyanaconda/ui/gui/spokes/welcome.glade:333
msgid "Type here to search."
msgstr "Scribe hic pro cercar."

#: pyanaconda/ui/gui/spokes/software_selection.glade:10
msgid "SOFTWARE SELECTION"
msgstr "SELECTION DE SOFTWARE"

#: pyanaconda/ui/gui/spokes/software_selection.glade:66
msgid "Base Environment"
msgstr "Ambiente basic"

#: pyanaconda/ui/gui/spokes/software_selection.glade:84
msgid "Add-Ons for Selected Environment"
msgstr "Additiones pro le ambiente seligite"

#: pyanaconda/ui/gui/spokes/network.glade:35
msgid "Add device"
msgstr "Adder un dispositivo"

#: pyanaconda/ui/gui/spokes/network.glade:49
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/network.glade:63
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr "_Adder"

#: pyanaconda/ui/gui/spokes/network.glade:92
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr "_Seliger le typo de dispositivo pro adder"

#: pyanaconda/ui/gui/spokes/network.glade:165
#: pyanaconda/ui/gui/spokes/network.glade:2287
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:204
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:417
msgid "Slaves"
msgstr "Sclavo"

#: pyanaconda/ui/gui/spokes/network.glade:441
#: pyanaconda/ui/gui/spokes/network.glade:840
#: pyanaconda/ui/gui/spokes/network.glade:1407
msgid "DNS"
msgstr "DNS"

#: pyanaconda/ui/gui/spokes/network.glade:471
#: pyanaconda/ui/gui/spokes/network.glade:828
#: pyanaconda/ui/gui/spokes/network.glade:1395
msgid "Default Route"
msgstr "Route predifinite"

#: pyanaconda/ui/gui/spokes/network.glade:554
#: pyanaconda/ui/gui/spokes/network.glade:792
#: pyanaconda/ui/gui/spokes/network.glade:1239
msgid "Speed"
msgstr "Velocitate"

#: pyanaconda/ui/gui/spokes/network.glade:579
#: pyanaconda/ui/gui/spokes/network.glade:780
msgid "Hardware Address"
msgstr "Adresse de apparato"

#: pyanaconda/ui/gui/spokes/network.glade:604
msgid "Parent"
msgstr "Parente"

#: pyanaconda/ui/gui/spokes/network.glade:616
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:664
msgctxt "GUI|Network|Wired"
msgid "_Configure..."
msgstr "_Configurar…"

#: pyanaconda/ui/gui/spokes/network.glade:953
msgid "Security"
msgstr "Securitate"

#: pyanaconda/ui/gui/spokes/network.glade:965
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr "Nomine del _rete"

#: pyanaconda/ui/gui/spokes/network.glade:979
msgid "Network Name"
msgstr "Nomine del rete"

#: pyanaconda/ui/gui/spokes/network.glade:991
msgid "Security Key"
msgstr "Clave de securitate"

#: pyanaconda/ui/gui/spokes/network.glade:1075
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr "_Usar como puncto de accesso..."

#: pyanaconda/ui/gui/spokes/network.glade:1092
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr "_Arrestar puncto de accesso..."

#: pyanaconda/ui/gui/spokes/network.glade:1109
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr "_Configurar..."

#: pyanaconda/ui/gui/spokes/network.glade:1227
msgid "IMEI"
msgstr "IMEI"

#: pyanaconda/ui/gui/spokes/network.glade:1251
msgid "Provider"
msgstr "Fornitor"

#: pyanaconda/ui/gui/spokes/network.glade:1434
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr "_Configurar..."

#: pyanaconda/ui/gui/spokes/network.glade:1544
msgid "VPN Type"
msgstr "Typo de VPN"

#: pyanaconda/ui/gui/spokes/network.glade:1556
msgid "Gateway"
msgstr "Passarella"

#: pyanaconda/ui/gui/spokes/network.glade:1568
msgid "Group Name"
msgstr "Nomine de gruppo"

#: pyanaconda/ui/gui/spokes/network.glade:1580
msgid "Group Password"
msgstr "Contrasigno de gruppo"

#: pyanaconda/ui/gui/spokes/network.glade:1592
#: pyanaconda/ui/gui/spokes/user.glade:117
msgid "User Name"
msgstr "Nomine de usator"

#: pyanaconda/ui/gui/spokes/network.glade:1700
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr "_Configurar..."

#: pyanaconda/ui/gui/spokes/network.glade:1822
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr "_Methodo"

#: pyanaconda/ui/gui/spokes/network.glade:1847
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr "URL de _configuration"

#: pyanaconda/ui/gui/spokes/network.glade:1872
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr "_Proxy HTTP"

#: pyanaconda/ui/gui/spokes/network.glade:1886
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr "Proxy H_TTPS"

#: pyanaconda/ui/gui/spokes/network.glade:1900
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr "Proxy _FTP"

#: pyanaconda/ui/gui/spokes/network.glade:1914
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr "Host _SOCKS"

#: pyanaconda/ui/gui/spokes/network.glade:2104
msgid "Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2120
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr "Disbloca_r"

#: pyanaconda/ui/gui/spokes/network.glade:2153
msgctxt "GUI|Network"
msgid "Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2173
msgid "More Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2193
msgctxt "GUI|Network"
msgid "_Host Name:"
msgstr "_Nomine de hospite:"

#: pyanaconda/ui/gui/spokes/network.glade:2222
msgctxt "GUI|Network|Hostname"
msgid "_Apply"
msgstr "_Applicar"

#: pyanaconda/ui/gui/spokes/network.glade:2226
msgid "Set as current system host name."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2241
msgid "Current host name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2293
msgid "NETWORK CONFIGURATION"
msgstr "CONFIGURATION DE RETE"

#: pyanaconda/ui/gui/spokes/network.glade:2338
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""
"Le accesso al rete es necessari pro obtener le information super le location "
"e pro facer actualisationes de software disponibile."

#: pyanaconda/ui/gui/spokes/network.glade:2368
msgid "Authentication"
msgstr "Authentication"

#: pyanaconda/ui/gui/spokes/network.glade:2382
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/network.glade:2397
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr "C_onnecter"

#: pyanaconda/ui/gui/spokes/network.glade:2469
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""
"Contrasignos o claves cryptographic es necessari\n"
"pro connecter se al rete sin filo"

#: pyanaconda/ui/gui/spokes/storage.glade:27
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Cancel & add more disks"
msgstr "_Cancellar e adder plus discos"

#: pyanaconda/ui/gui/spokes/storage.glade:43
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr "_Recuperar spatio"

#: pyanaconda/ui/gui/spokes/storage.glade:77
#: pyanaconda/ui/gui/spokes/storage.glade:295
msgid "INSTALLATION OPTIONS"
msgstr "OPTIONES DE INSTALLATION"

#: pyanaconda/ui/gui/spokes/storage.glade:94
#: pyanaconda/ui/gui/spokes/storage.glade:311
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""
"Le amonta de spacio necessari pro installar programma seligite essera "
"monstrate hic."

#: pyanaconda/ui/gui/spokes/storage.glade:120
#: pyanaconda/ui/gui/spokes/storage.glade:337
msgid "disk free"
msgstr "disco libere"

#: pyanaconda/ui/gui/spokes/storage.glade:136
#: pyanaconda/ui/gui/spokes/storage.glade:352
msgid "Free space available for use."
msgstr "Spatio libere disponibile pro usage."

#: pyanaconda/ui/gui/spokes/storage.glade:152
#: pyanaconda/ui/gui/spokes/storage.glade:368
msgid "fs free"
msgstr "systema de files libere"

#: pyanaconda/ui/gui/spokes/storage.glade:168
#: pyanaconda/ui/gui/spokes/storage.glade:384
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""
"Spatio libere que es indisponibile ma recuperabile del partitiones existente."

#: pyanaconda/ui/gui/spokes/storage.glade:191
#: pyanaconda/ui/gui/spokes/storage.glade:407
msgid "Here we'll describe what your options are."
msgstr "Hic nos describera que es le optiones."

#: pyanaconda/ui/gui/spokes/storage.glade:222
msgid "Need Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:245
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr "_Quitar le installator"

#: pyanaconda/ui/gui/spokes/storage.glade:261
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/storage.glade:438
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:450
#: pyanaconda/ui/gui/spokes/storage.glade:1136
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:67
msgid "INSTALLATION DESTINATION"
msgstr "DESTINATION DEL INSTALLATION"

#: pyanaconda/ui/gui/spokes/storage.glade:512
msgid "Device Selection"
msgstr "Selection de dispositivo"

#: pyanaconda/ui/gui/spokes/storage.glade:531
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""
"Selige le dispositivo(s) pro le installation. Illos non essera toccate usque "
"al clic sur le button «Comenciar le installation» in le menu principal."

#: pyanaconda/ui/gui/spokes/storage.glade:550
msgid "Local Standard Disks"
msgstr "Discos normal local"

#: pyanaconda/ui/gui/spokes/storage.glade:614
#: pyanaconda/ui/gui/spokes/storage.glade:754
msgid "Disks left unselected here will not be touched."
msgstr "Le discos restate non seligite hic non essera toccate."

#: pyanaconda/ui/gui/spokes/storage.glade:633
msgid "Specialized & Network Disks"
msgstr "Discos specialisate e de rete"

#: pyanaconda/ui/gui/spokes/storage.glade:704
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr "_Adder un disco..."

#: pyanaconda/ui/gui/spokes/storage.glade:721
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:781
msgid "Storage Configuration"
msgstr "Configuration de immagazinage"

#: pyanaconda/ui/gui/spokes/storage.glade:801
msgctxt "GUI|Storage"
msgid "A_utomatic"
msgstr "A_utomatic"

#: pyanaconda/ui/gui/spokes/storage.glade:819
msgctxt "GUI|Storage"
msgid "_Custom"
msgstr "_Personalisate"

#: pyanaconda/ui/gui/spokes/storage.glade:838
msgctxt "GUI|Storage"
msgid "Advanced Custom (_Blivet-GUI)"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:869
msgctxt "GUI|Storage"
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:901
msgid "Encryption"
msgstr "Cryptation"

#: pyanaconda/ui/gui/spokes/storage.glade:931
msgctxt "GUI|Storage"
msgid "_Encrypt my data."
msgstr "_Cryptar mi datos."

#: pyanaconda/ui/gui/spokes/storage.glade:947
msgid "You'll set a passphrase next."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:997
msgid "Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:1028
msgid "summary"
msgstr "summario"

#: pyanaconda/ui/gui/spokes/storage.glade:1061
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:1100
msgctxt "GUI|Storage"
msgid "_Refresh..."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:351
msgid "CREATE USER"
msgstr "CREAR UN USATOR"

#: pyanaconda/ui/gui/spokes/user.glade:61
msgctxt "GUI|User"
msgid "_Full name"
msgstr "No_mine complete"

#: pyanaconda/ui/gui/spokes/user.glade:79
msgctxt "GUI|User"
msgid "_User name"
msgstr "Nomine de _usator"

#: pyanaconda/ui/gui/spokes/user.glade:100
msgid "Full Name"
msgstr "Nomine complete"

#: pyanaconda/ui/gui/spokes/user.glade:131
msgctxt "GUI|User"
msgid "_Password"
msgstr "Contra_signo"

#: pyanaconda/ui/gui/spokes/user.glade:149
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr "_Confirmar le contrasigno"

#: pyanaconda/ui/gui/spokes/user.glade:191
#: pyanaconda/ui/gui/spokes/root_password.glade:107
msgid "Confirm Password"
msgstr "Confirmar contrasigno"

#: pyanaconda/ui/gui/spokes/user.glade:204
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:215
msgctxt "GUI|User"
msgid "_Require a password to use this account"
msgstr "_Requirer un contrasigno pro usar iste conto"

#: pyanaconda/ui/gui/spokes/user.glade:255
#: pyanaconda/ui/gui/spokes/root_password.glade:142
msgid "empty password"
msgstr "contrasigno vacue"

#: pyanaconda/ui/gui/spokes/user.glade:274
msgctxt "GUI|User"
msgid "_Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:294
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr "_Avantiate..."

#: pyanaconda/ui/gui/spokes/custom_storage.glade:175
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:176
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr "Adder"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:191
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:192
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr "Remover"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:207
msgid "Reload storage configuration from disk."
msgstr "Recargar le configuration de immagazinage del disco."

#: pyanaconda/ui/gui/spokes/custom_storage.glade:208
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:273
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1076
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1221
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1308
msgid "Selected Device"
msgstr "Dispositivo seligite"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:291
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1094
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1239
#: pyanaconda/ui/gui/spokes/custom_storage.glade:1326
msgid "Device description"
msgstr "Description de dispositivo"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:352
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "Mount _Point:"
msgstr "_Puncto de montage:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:399
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "Desired _Capacity:"
msgstr "_Capacitate desirate:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:450
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Label:"
msgstr "_Etiquetta:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:497
msgid "Device(s):"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:526
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr "_Modificar…"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:558
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Name:"
msgstr "_Nomine:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:613
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr "_Typo de dispositivo:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:656
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr "_Cryptar"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:697
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr "S_ystema de files:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:740
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr "Re_formatar"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:795
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr "Gruppo de _volumines:"

#. check_accelerators: configureButton
#: pyanaconda/ui/gui/spokes/custom_storage.glade:848
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr "_Modificar..."

#: pyanaconda/ui/gui/spokes/custom_storage.glade:880
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr "Nivello de RA_ID:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:937
msgctxt "GUI|Custom Partitioning|Configure"
msgid "LUKS Version:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1008
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Update Settings"
msgstr "Parametros de act_ualisation"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1031
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1128
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""
"Iste dispositivo es cryptate e non pote esser legite sin un phrase de "
"contrasigno valide. Tu pote disblocar lo in basso."

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1149
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr "_Phrase de contrasigno:"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1178
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr "_Disblocar"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1273
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""
"Iste dispositivo non pote esser editate directemente. On pote remover lo o "
"seliger un altere dispositivo."

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1421
msgid "AVAILABLE SPACE"
msgstr "SPATIO DISPONIBILE"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1474
msgid "TOTAL SPACE"
msgstr "SPATIO TOTAL"

#: pyanaconda/ui/gui/spokes/custom_storage.glade:1514
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr "_Reinitialisar toto"

#: pyanaconda/ui/gui/spokes/installation_source.glade:29
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/installation_source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:95
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr "Pr_este"

#: pyanaconda/ui/gui/spokes/installation_source.glade:128
msgid "MEDIA VERIFICATION"
msgstr "VERIFICATION DE MEDIO"

#: pyanaconda/ui/gui/spokes/installation_source.glade:155
msgid "Verifying media, please wait..."
msgstr "Verification del medio, attende..."

#: pyanaconda/ui/gui/spokes/installation_source.glade:224
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/installation_source.glade:238
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr "_OK"

#: pyanaconda/ui/gui/spokes/installation_source.glade:267
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr "Activar proxy _HTTP"

#: pyanaconda/ui/gui/spokes/installation_source.glade:299
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy Host:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:316
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr "<span size=\"small\"><b>Exemplo:</b> squid.mysite.org:3128</span>"

#: pyanaconda/ui/gui/spokes/installation_source.glade:349
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr "_Usar authentication"

#: pyanaconda/ui/gui/spokes/installation_source.glade:375
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name:"
msgstr "_Nomine de usator:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:392
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word:"
msgstr "Contra_signo:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:474
msgid "INSTALLATION SOURCE"
msgstr "FONTE DE INSTALLATION"

#: pyanaconda/ui/gui/spokes/installation_source.glade:531
msgid "Which installation source would you like to use?"
msgstr "Qual fonte de installation debe usar se?"

#: pyanaconda/ui/gui/spokes/installation_source.glade:545
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr "Medio de installation _autodetegite:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:571
msgid "Device:"
msgstr "Dispositivo:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:583
msgid "Label:"
msgstr "Etiquetta:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:593
#: pyanaconda/ui/gui/spokes/installation_source.glade:710
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr "_Verificar"

#: pyanaconda/ui/gui/spokes/installation_source.glade:617
msgctxt "GUI|Software Source"
msgid "Installation media via SE/_HMC"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:635
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr "File _ISO:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:663
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr "D_ispositivo:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:694
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr "_Eliger un ISO"

#: pyanaconda/ui/gui/spokes/installation_source.glade:734
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr "_In le rete:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:798
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr "C_onfiguration del proxy..."

#: pyanaconda/ui/gui/spokes/installation_source.glade:815
#: pyanaconda/ui/gui/spokes/installation_source.glade:1295
msgctxt "GUI|Software Source"
msgid "URL type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:830
#: pyanaconda/ui/gui/spokes/installation_source.glade:1280
msgid "repository URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:831
#: pyanaconda/ui/gui/spokes/installation_source.glade:1281
msgid "mirrorlist"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:832
#: pyanaconda/ui/gui/spokes/installation_source.glade:1282
msgid "metalink"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:859
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr "Optiones de montage N_FS:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:874
#: pyanaconda/ui/gui/spokes/installation_source.glade:875
msgid "This field is optional."
msgstr "Iste campo es optional."

#: pyanaconda/ui/gui/spokes/installation_source.glade:908
msgid "Updates"
msgstr "Actualisationes"

#: pyanaconda/ui/gui/spokes/installation_source.glade:922
msgctxt "GUI|Software Source"
msgid "Install the _latest available software updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:939
msgctxt "GUI|Software Source"
msgid ""
"Install the _default versions provided by the installation source (above) "
"only"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:967
msgid "Additional repositories"
msgstr "Repositorios additional"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1008
msgid "Enabled"
msgstr "Activate"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1053
#: pyanaconda/ui/gui/spokes/installation_source.glade:1054
msgid "Add a new repository."
msgstr "Adder un nove repositorio."

#: pyanaconda/ui/gui/spokes/installation_source.glade:1055
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr "A_dder"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1072
#: pyanaconda/ui/gui/spokes/installation_source.glade:1073
msgid "Remove the selected repository."
msgstr "Remover le repositorio seligite."

#: pyanaconda/ui/gui/spokes/installation_source.glade:1074
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr "_Remover"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1091
#: pyanaconda/ui/gui/spokes/installation_source.glade:1092
msgid "Revert to the previous list of repositories."
msgstr "Revenir al previe lista de repositorios."

#: pyanaconda/ui/gui/spokes/installation_source.glade:1093
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr "Reini_tialisar"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1123
#: pyanaconda/ui/gui/spokes/installation_source.glade:1124
msgid "URL for the repository, without protocol."
msgstr "URL pro le repositorio, sin protocollo."

#: pyanaconda/ui/gui/spokes/installation_source.glade:1136
#: pyanaconda/ui/gui/spokes/installation_source.glade:1137
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr "URL de proxy in le forma protocollo://host:[porto]"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1149
#: pyanaconda/ui/gui/spokes/installation_source.glade:1150
msgid "Optional proxy user name."
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1162
msgid "Optional proxy password."
msgstr "Contrasigno optional al proxy."

#: pyanaconda/ui/gui/spokes/installation_source.glade:1176
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr "No_mine:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1193
#: pyanaconda/ui/gui/spokes/installation_source.glade:1194
msgid "Protocol for the repository URL."
msgstr "Protocollo pro le URL de repositorio."

#: pyanaconda/ui/gui/spokes/installation_source.glade:1213
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr "URL de pro_xy:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1230
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_source.glade:1247
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr "Contra_signo:"

#: pyanaconda/ui/gui/spokes/installation_source.glade:1264
#: pyanaconda/ui/gui/spokes/installation_source.glade:1265
msgid "Name of the repository."
msgstr "Nomine del repositorio."

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:136
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr "_Cercar per:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:153
msgid "Port / Target / LUN #"
msgstr "Porto / Target / LUN #"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:154
msgid "Target WWID"
msgstr "WWID de scopo"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:193
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr "_Porto:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:219
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr "_Scopo:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:247
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr "_LUN:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:289
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr "_WWID:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:339
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search Res_ults:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:400
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:597
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:800
msgid "WWID"
msgstr "WWID"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:428
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:596
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:842
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:930
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1176
msgid "Interconnect"
msgstr "Interconnexion"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:442
msgid "Model"
msgstr "Modello"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:456
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1541
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1817
msgid "LUN"
msgstr "LUN"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:470
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1831
msgid "Port"
msgstr "Porto"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:484
msgid "Target"
msgstr "Scopo"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:498
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:595
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:828
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:929
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1162
msgid "Vendor"
msgstr "Venditor"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:512
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1249
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1424
msgid "Namespace"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:526
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1250
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1452
msgid "Mode"
msgstr "Modo"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:553
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr "_Cercar"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:578
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:635
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr "_Monstrar solmente dispositivos de:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:676
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr "_Monstrar solmente dispositivos con:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:717
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr "_Monstrar solmente dispositivos qual contine:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:856
msgid "Paths"
msgstr "Percursos"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:886
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr "Dispositivos _multivie"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:912
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:931
msgid "ID"
msgstr "ID"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:969
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr "_Monstrar solmente dispositivos de:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1010
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr "_Monstrar solmente dispositivos con:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1051
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr "_Monstrar solmente dispositivos qual contine:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1134
msgid "Identifier"
msgstr "Identificator"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1206
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr "_Altere dispositivos SAN"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1232
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1288
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices|Mode"
msgid "Show Only Devices in _Mode:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1305
msgid "Sector"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1306
msgid "Memory"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1333
msgctxt "GUI|Installation Destination|Filter|NVDIMM Devices|Namespace"
msgid "_Namespace:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1496
msgctxt "GUI|Installation Destination|Filter"
msgid "_NVDIMM Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1522
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1539
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1747
msgid "CCW"
msgstr "CCW"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1540
#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1803
msgid "WWPN"
msgstr "WWPN"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1578
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr "_CCW:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1621
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr "_WWPN:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1664
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr "_LUN:"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1861
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr "Dispositivos _zSeries"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1884
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1899
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1914
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr "Adder scopo _iSCSI…"

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1929
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr "Adder SAN FCo_E..."

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1944
msgctxt "GUI|Installation Destination|Filter"
msgid "Reconfigure _NVDIMM..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_storage.glade:1959
msgctxt "GUI|Installation Destination|Filter"
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr "_Cancellar"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr "_Salveguardar modificationes"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:82
msgid "ADVANCED USER CONFIGURATION"
msgstr "CONFIGURATION AVANTIATE DE USATORES"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:105
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr "_Directorio personal:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:140
msgid "User and Group IDs"
msgstr "Identificatores de usator e gruppo"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:160
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr "Specificar manualmente un identificator de _usator:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:177
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr "Specificar manualmente un identificator de _gruppo:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:237
msgid "Group Membership"
msgstr "Adhesion de gruppo"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:265
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr "_Adder le usator al sequente gruppos:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:298
msgid "Example:"
msgstr "Exemplo:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:315
msgid "wheel, my-team (1245), project-x (29935)"
msgstr "wheel, my-team (1245), project-x (29935)"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:348
msgid "Tip:"
msgstr "Aviso:"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:367
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.glade:8
msgid "BLIVET GUI PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.glade:131
msgctxt "GUI|Blivet Gui Partitioning"
msgid "_Undo last action"
msgstr ""

#: pyanaconda/ui/gui/spokes/blivet_gui.glade:150
msgctxt "GUI|Blivet Gui Partitioning"
msgid "_Reset All"
msgstr "_Reinitialisar toto"

#: pyanaconda/ui/gui/spokes/installation_progress.glade:8
#: pyanaconda/ui/gui/spokes/installation_progress.glade:164
msgid "INSTALLATION PROGRESS"
msgstr ""

#: pyanaconda/ui/gui/spokes/installation_progress.glade:75
msgid "Preparing to install"
msgstr "Preparation de installation"

#: pyanaconda/ui/gui/spokes/welcome.glade:23
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "I want to _exit."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:37
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "I want to _proceed."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr "Iste software es instabile, in stato de pre-edition."

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"Notice: This is pre-released software that is intended for development and "
"testing purposes only.  Do *not* use this software for any critical work or "
"for production environments.\n"
"\n"
"By clicking \"I want to proceed\", you understand and accept the risks "
"associated with pre-released software, that you intend to use this for "
"testing and development purposes only and are willing to report any bugs or "
"issues in order to enhance this work.\n"
"\n"
"If you do not understand or accept the risks, then please exit this program "
"by selecting \"I want to exit\" which will reboot your system."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:130
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:228
msgid "What language would you like to use during the installation process?"
msgstr "Que lingua debe esser usate durante le processo de installation?"

#: pyanaconda/ui/gui/spokes/welcome.glade:313
msgid "Languages"
msgstr "Linguas"

#: pyanaconda/ui/gui/spokes/welcome.glade:385
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:419
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:8
#: pyanaconda/ui/gui/spokes/root_password.glade:229
msgid "ROOT PASSWORD"
msgstr "CONTRASIGNO DE SUPERUSATOR"

#: pyanaconda/ui/gui/spokes/root_password.glade:53
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr "Contrasigno del supe_rusator:"

#: pyanaconda/ui/gui/spokes/root_password.glade:67
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr "_Confirmar:"

#: pyanaconda/ui/gui/spokes/root_password.glade:163
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"Le conto del superusator root se utilisa pro administrar le systema. "
"Introduce un contrasigno pro le usator root."

#: pyanaconda/ui/gui/spokes/root_password.glade:184
msgid "Lock root account"
msgstr ""

#: pyanaconda/ui/gui/spokes/root_password.glade:200
msgid "Allow root SSH login with password"
msgstr ""

#: pyanaconda/ui/gui/main.glade:24
msgctxt "GUI|Error Dialog"
msgid "_Exit Installer"
msgstr "_Exir del installator"

#: pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr "Error"

#: pyanaconda/ui/gui/main.glade:94
msgctxt "GUI|Main"
msgid "_No"
msgstr "_No"

#: pyanaconda/ui/gui/main.glade:108
msgctxt "GUI|Main"
msgid "_Yes"
msgstr "_Si"

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""
"Es tu secur que tu vole quitar le\n"
"processo de installation?"

#: pyanaconda/ui/gui/main.glade:175
msgid "Quit"
msgstr "Quitar"

#: data/liveinst/gnome/fedora-welcome.desktop:3
#: data/liveinst/gnome/fedora-welcome.js:62
msgid "Welcome to Fedora"
msgstr "Benvenite a Fedora"

#: data/liveinst/liveinst.desktop:3
msgid "Install to Hard Drive"
msgstr "Installar sur le disco dur"

#: data/liveinst/liveinst.desktop:4
msgid "Install"
msgstr "Installar"

#: data/liveinst/liveinst.desktop:5
msgid "Install the live CD to your hard disk"
msgstr "Installar le LiveCD sur le disco dur"

#. TRANSLATORS: Icon name, probably should not be translated
#: data/liveinst/liveinst.desktop:11
msgid "org.fedoraproject.AnacondaInstaller"
msgstr ""

#: data/liveinst/liveinst-setup.desktop:4
msgid "Liveinst Setup"
msgstr ""

#: data/liveinst/gnome/fedora-welcome.js:91
msgid "Try Fedora"
msgstr "Probar Fedora"

#: data/liveinst/gnome/fedora-welcome.js:107
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""
"Actualmente Fedora es lanceate desde un live-medio. \n"
"Il pote installar Fedora ora, o eliger \"Installar sur le disco dur\" in le "
"Resume de activitates plus tarde."

#: data/liveinst/gnome/fedora-welcome.js:127
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""
"Il pote eliger \"Installar sur le disco dur\"\n"
"in le Resume de activitates plus tarde."

#: data/liveinst/gnome/fedora-welcome.js:130
msgid "Close"
msgstr "Clauder"

#: ../data/liveinst/console.apps/liveinst.h:1
msgid "Starting Install to Hard Drive"
msgstr "Initio de installation sur le disco dur"

#~ msgid ""
#~ "The following errors occurred with your partitioning:\n"
#~ "\n"
#~ "%(errortxt)s\n"
#~ "\n"
#~ "The installation will now terminate."
#~ msgstr ""
#~ "Le sequente errores occurreva durante partitionamento:\n"
#~ "\n"
#~ "%(errortxt)s\n"
#~ "\n"
#~ "Ora le installation essera terminate."

#~ msgid ""
#~ "An error has occurred - no valid devices were found on which to create "
#~ "new file systems.  Please check your hardware for the cause of this "
#~ "problem."
#~ msgstr ""
#~ "Un error ha occurrite - necun dispositivo valide esseva trovate pro crear "
#~ "nove systemas de files. Examina le hardware pro le causa de iste problema."

#~ msgid "Generating updated storage configuration"
#~ msgstr "Generation del configuration de immagazinage actualisate"

#~ msgid "storage configuration failed: %s"
#~ msgstr "insuccesso del configuration de immagazinage: %s"

#~ msgid "Volume Group name %s is already in use. Not saving changes."
#~ msgstr ""
#~ "Le nomine de gruppo de volumines %s es jam in uso. Le cambiamentos non "
#~ "essera servate."
